# Weekly Papers
This project automatically fetches the latest papers from ChemRxiv based on categories.

Only the most recent articles for each category are retained, up to a maximum of 100 papers.

You can click the "Watch" button to receive daily email notifications.

Last update: 2025-09-08

| **Title** | **Date** | **Doi** | **Abstract** | **Categories** | **Suppitems** |
| --- | --- | --- | --- | --- | --- |
| **[Exploring the Potential of an End-to-End AI Design Framework for Photoresponsive Drug Delivery.](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b6acc823be8e43d6777101/original/exploring-the-potential-of-an-end-to-end-ai-design-framework-for-photoresponsive-drug-delivery.pdf)** | 2025-09-05 | 10.26434/chemrxiv-2025-9dgq3 | <details><summary>Show</summary><p>In our effort to more reliably leverage large language models for designing effective photoresponsive molecular materials for drug delivery, we developed the Explanation CPO (XCPO), an end-to-end reinforcement learning fine-tuning workflow. The maturity of RLHF techniques has greatly stimulated the flourishing application of general-purpose LLMs across various scientific domains. Compared with representative RLHF approaches like CPO, our XCPO emphasizes the potential and effectiveness of a fully AI-driven, end-to-end fine-tuning process. In this workflow, the preference dataset for RLFT is derived from the combined output of AI models and XAI tools, whereas traditional RLHF pipelines require domain researchers to identify key features based on their expertise. This streamlined improvement via integration with XAI tools has not been previously reported. Based on our machine learning evaluation and computational chemistry validation, our end-to-end XCPO approach outperformed CPO by over 27\% on the MoE-based Qwen3Moe architecture, while the designed photoresponsive molecular materials aligned well with chemical intuition. Our end-to-end workflow compensates for the subjective influence of chemists' intuitive experience, enabling the model to iterate within a fully AI-driven process.</p></details> | Theoretical and Computational Chemistry; Biological and Medicinal Chemistry; Materials Science; Computational Chemistry and Modeling; Machine Learning; Artificial Intelligence | <details><summary>Show</summary><p>[]</p></details> |
| **[Distilling Knowledge from Catalysis Literature with Long-Context LLM Agents](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b64a88a94eede1540252d7/original/distilling-knowledge-from-catalysis-literature-with-long-context-llm-agents.pdf)** | 2025-09-05 | 10.26434/chemrxiv-2025-925dv-v2 | <details><summary>Show</summary><p>Decades of catalysis knowledge remain locked in unstructured prose, hindering data-driven discovery. Existing text-mining tools struggle to establish the synthesis-structure-performance relationships critical for catalyst knowledge discovery, as they rarely connect synthesis protocols in one section with the resulting material properties and performance outcomes reported elsewhere. Here, we present CATDA (Corpus-aware Automated Text-to-Graph Catalyst Discovery Agent), a long-context large language model (LLM)–driven agentic framework that reads full documents and distills them into actionable, provenance–tracked knowledge graphs linking material properties, multi-step synthesis, conditions, and testing outcomes. Applied at corpus scale, CATDA extracts data with near-human fidelity (F1 = 0.983) and a 12-fold speedup over manual curation. This structured knowledge is made accessible through two synergistic applications: a DatasetAgent for exporting machine-learning-ready tables, and a CatAgent providing a conversational, citation-linked interface for interactive discovery. The high-quality dataset enabled the training of a predictive model for ethylbenzene conversion, while simultaneously exposing systemic challenges such as feature sparsity and protocol heterogeneity in the source literature. By transforming the literature into a queryable and computable resource, CATDA offers a scalable route to accelerate large-scale data analysis, quantitative modeling, and rational catalyst design paradigm.</p></details> | Theoretical and Computational Chemistry; Artificial Intelligence | <details><summary>Show</summary><p>[{"title": "Supplementary Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b7ff81728bf9025ef7bef3/original/supplementary-information.docx"}]</p></details> |
| **[LightChem: A Lightweight Domain-Specific Language Model for Molecular Property and Reaction Prediction in Chemistry](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b4709e728bf9025e703bce/original/light-chem-a-lightweight-domain-specific-language-model-for-molecular-property-and-reaction-prediction-in-chemistry.pdf)** | 2025-09-03 | 10.26434/chemrxiv-2025-wd80w | <details><summary>Show</summary><p>Large language models (LLMs) have achieved breakthroughs in natural language processing, yet their use in chemistry is limited by insufficient domain knowledge, high computational demands, and poor scalability. To overcome these challenges, we developed LightChem, a lightweight model that integrates retrieval-augmented generation (RAG) and tool-calling capabilities, providing a platform driven by chemical knowledge and theoretical calculations. LightChem adopts a three-layer architecture, data aggregation, intelligent retrieval, and specialized tool integration, supported by the self-developed RAPTOR system for real-time literature updates. The platform incorporates a suite of computational modules, including automated molecular property prediction (e.g., PoLogP), retrosynthetic planning via ReSynZ, and scalable quantum chemistry packages such as CIM and GEBF, enabling high-accuracy simulations of complex systems. Case studies demonstrate successful predictions of lipophilicity, excitation/emission spectra, and synthetic pathways for drug-like molecules, while large-scale calculations reproduce binding energies in zeolite clusters and excitation energies in GFP chromophores with accuracy comparable to experimental values. Beyond core computations, LightChem features a user-friendly web interface that connects sophisticated algorithms with practical laboratory workflows. It provides modules for reagent management, synthesis assessment, safety evaluation, and experimental design, thereby lowering the barrier for non-specialists and supporting systematic research in photosensitive and functional molecules. Applications to gold nanoclusters and zeolites further highlight the platform’s potential in photocatalysis and optical materials, while also revealing current limitations in predicting HOMO–LUMO gaps and optical properties of complex aggregates. LightChem demonstrates good performance in molecular property prediction, retrosynthetic planning, and large-scale quantum simulations, making it a versatile assistant for chemical research, bridging knowledge-driven reasoning with first-principles accuracy.</p></details> | Theoretical and Computational Chemistry; Organic Chemistry; Materials Science; Machine Learning; Artificial Intelligence | <details><summary>Show</summary><p>[{"title": "LightChem: A Lightweight Domain-Specific Language Model for Molecular Property and Reaction Prediction in Chemistry", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b4842723be8e43d6268933/original/light-chem-a-lightweight-domain-specific-language-model-for-molecular-property-and-reaction-prediction-in-chemistry.pdf"}]</p></details> |
| **[The elephant in the lab: Synthesizability in generative small-molecule design](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b1ed3623be8e43d6d76098/original/the-elephant-in-the-lab-synthesizability-in-generative-small-molecule-design.pdf)** | 2025-09-02 | 10.26434/chemrxiv-2025-1lcpq | <details><summary>Show</summary><p>The design of small molecules with tailored properties is a central goal in chemistry and materials science. Recent advances in machine learning provide powerful tools to accelerate the pace of discovery. One promising avenue for acceleration involves the use of generative models that propose novel candidates for diverse optimization tasks. Despite their promise, these methods are often evaluated solely using computational benchmarks, and many studies fail to advance proposed candidates to experimental validation in the wet lab. A key reason for this gap, the elephant in the room, is the limited synthesizability of the generated molecules. In response, the community has recently developed various strategies to address this challenge and incorporate synthesizability into generative design workflows. In this opinion, we provide a comprehensive overview of recent contributions that explicitly tackle molecular synthesizability, highlighting notable advances. We also discuss key limitations of current approaches and outline promising directions for future research.</p></details> | Theoretical and Computational Chemistry; Organic Chemistry; Organic Synthesis and Reactions; Machine Learning; Artificial Intelligence | <details><summary>Show</summary><p>[]</p></details> |
| **[A Computational Community Blind Challenge on Pan-Coronavirus Drug Discovery Data](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b4de6623be8e43d633cf11/original/a-computational-community-blind-challenge-on-pan-coronavirus-drug-discovery-data.pdf)** | 2025-09-02 | 10.26434/chemrxiv-2025-zd9mr-v3 | <details><summary>Show</summary><p>Computational blind challenges offer critical, unbiased assessment opportunities to assess and accelerate scientific progress, as demonstrated by a breadth of breakthroughs over the last decade. We report the outcomes and key insights from an open science community blind challenge focused on computational methods in drug discovery, using lead optimization data from the AI-driven Structure-enabled Antiviral Platform (ASAP) Discovery Consortium’s pan-coronavirus antiviral discovery program, in partnership with Polaris and the OpenADMET project. This collaborative initiative invited global participants from both academia and industry to develop and apply computational methods to predict the biochemical potency and crystallographic ligand poses of small molecules against key coronavirus targets, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) and Middle East Respiratory Syndrome Coronavirus (MERS-CoV) main protease (Mpro), as well as multiple ADMET assay endpoints, using previously undisclosed comprehensive experimental drug discovery datasets as benchmarks. By evaluating submissions across multiple tasks and compounds, we established performance leaderboards and conducted meta-analyses to assess methodological strengths, common pitfalls, and areas for improvement. This analysis provides a foundation for best practices in real-world machine learning evaluation, grounded in community-driven benchmarking. We also highlight how next-generation platforms, such as Polaris, enable rigorous challenge design, embedded evaluation frameworks, and broad community engagement. This paper reports the collective findings of the challenge, offering a high-level overview of the data, evaluation infrastructure, and top- performing strategies. We further provide context and support for the accompanying papers authored by the challenge participants in this special issue, which explore individual approaches in greater depth. Together, these contributions aim to advance reproducible, trustworthy, and high-impact computational methods in drug discovery, and to explore best practices and pitfalls in future blind challenge design and execution, including planned initiatives for the OpenADMET project.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Artificial Intelligence | <details><summary>Show</summary><p>[{"title": "Supplementary Figures", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b4de6623be8e43d633cf97/original/supplementary-figures.pdf"}]</p></details> |
| **[Explainable GNN Framework Guided by Local Chemical Features to Predict Binding Energies in Bimetallic Alloys](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b7370723be8e43d68bbfd6/original/explainable-gnn-framework-guided-by-local-chemical-features-to-predict-binding-energies-in-bimetallic-alloys.pdf)** | 2025-09-05 | 10.26434/chemrxiv-2025-s2p1s | <details><summary>Show</summary><p>Adsorption energies are key catalytic descriptors that reveal adsorbate-site interactions on heterogeneous catalysts. However, their computation via DFT is time-consuming, limiting high-throughput screening. This work presents a machine learning (ML) methodology based on graph representations of local adsorption sites, using a Graph Neural Network (GNN) with per-atom local descriptors derived from accessible physicochemical properties. The approach is evaluated on two bimetallic datasets. The first includes AB-type bimetallic flat surfaces with varying A:B ratios, predicting binding energies for small monodentate adsorbates (C, N, O, S, H) with MSEs of 0.073/0.181 eV (train/test). The second dataset comprises reaction energies of key intermediates for CO2 hydrogenation on Ni-Ga-based surfaces. The GNN model achieves an impressive performance (MSE: 0.001/0.002 (train/test) eV) on complex atomic configurations, even bidentate ones. Beyond predictive performance, clustering analysis provides an explainable framework, showing how structural and electronic descriptors can rationally guide catalyst design and deepen understanding of adsorbate-metal interactions.</p></details> | Theoretical and Computational Chemistry; Catalysis; Theory - Computational; Machine Learning; Heterogeneous Catalysis; Materials Chemistry | <details><summary>Show</summary><p>[{"title": "Explainable GNN Framework Guided by Local Chemical Features to Predict Binding Energies in Bimetallic Alloys", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b73fe4a94eede154278666/original/explainable-gnn-framework-guided-by-local-chemical-features-to-predict-binding-energies-in-bimetallic-alloys.pdf"}]</p></details> |
| **[epiGPTope: A machine learning-based epitope generator and classifier](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b5d50d23be8e43d655b9b0/original/epi-gp-tope-a-machine-learning-based-epitope-generator-and-classifier.pdf)** | 2025-09-04 | 10.26434/chemrxiv-2025-hjjjb | <details><summary>Show</summary><p>Epitopes are short antigenic peptide sequences which are recognized by antibodies or immune cell receptors. These are central to the development of immunotherapies, vaccines, and diagnostics. However, the rational design of synthetic epitope libraries is challenging due to the large combinatorial sequence space, 20n combinations for linear epitopes of n amino acids, making screening and testing unfeasible, even with high throughput experimental techniques. In this study, we present a large language model, epiGPTope, pre-trained on protein data and specifically fine-tuned on linear epitopes, which for the first time can directly generate novel epitope-like sequences, which are found to possess statistical properties analogous to the ones of known epitopes. This generative approach can be used to prepare libraries of epitope candidate sequences. We further train statistical classifiers to predict whether an epitope sequence is of bacterial or viral origin, thus narrowing the candidate library and increasing the likelihood of identifying specific epitopes. We propose that such combination of generative and predictive models can be of assistance in epitope discovery. The approach uses only primary amino acid sequences of linear epitopes, bypassing the need for a geometric framework or hand-crafted features of the sequences. By developing a method to create biologically feasible sequences, we anticipate faster and more cost-effective generation and screening of synthetic epitopes, with relevant applications in the development of new biotechnologies.</p></details> | Theoretical and Computational Chemistry; Biological and Medicinal Chemistry; Biochemistry; Machine Learning | <details><summary>Show</summary><p>[{"title": "Supplementary information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b5d9aca94eede154f07470/original/supplementary-information.pdf"}]</p></details> |
| **[Activity Descriptors of Mo2C-based Catalysts for C-OH Bond Activation](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b58a4d23be8e43d64c11fa/original/activity-descriptors-of-mo2c-based-catalysts-for-c-oh-bond-activation.pdf)** | 2025-09-04 | 10.26434/chemrxiv-2025-pg52l | <details><summary>Show</summary><p>Deriving structure-activity relationships is crucial for designing efficient catalysts. To aid in this quest, data-driven methods such as machine learning (ML) are emerging. In this work, we incorporate ML tools with accurate density functional theory (DFT) energetics, electronic and geometric features, in the context of designing efficient molybdenum carbide (Mo2C)-based catalysts for biomass conversion. Previously, it was shown that C-OH activation is the rate-determining step in the hydrodeoxygenation (HDO) reaction. Therefore, in this work, DFT was used to obtain accurate barriers (Ea) and reaction energy (∆E) for the C-OH activation in HDO over the most stable (111) and meta-stable facets (010, 101, and 110) of Mo2C and transition metal doped Mo2C. The 101 facet was identified as the most active facet for C-OH activation. While, doping of the active site with Zr and Nb was identified as promising strategy to improve the activity. Further, scikit-learn’s ML models were used to obtain the best primary features correlating with the Ea. Ridge Regression (RR) gives the best ML model for predicting Ea with a test RMSE of 0.21 eV. SHapley Additive exPlanations (SHAP) analysis was then performed which reveals that ∆E and d-band center are the most important features contributing to the activity. Finally, SISSO was used to validate RR model and SHAP results, and to obtain a 2-dimensional physically interpretable generic-descriptor comprising of dopant’s local environment, d-band features, and ∆E.</p></details> | Theoretical and Computational Chemistry; Materials Science; Catalysis; Computational Chemistry and Modeling; Machine Learning; Heterogeneous Catalysis | <details><summary>Show</summary><p>[{"title": "cif files for slab model", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b58de5728bf9025e9a002a/original/cif-files-for-slab-model.zip"}]</p></details> |
| **[Mapping Antibiotic Photocatalytic Transformation and Resistance Risks with a DFT-Informed Machine Learning Workflow](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b54cf2728bf9025e8de716/original/mapping-antibiotic-photocatalytic-transformation-and-resistance-risks-with-a-dft-informed-machine-learning-workflow.pdf)** | 2025-09-04 | 10.26434/chemrxiv-2025-qwcm6 | <details><summary>Show</summary><p>The photocatalytic degradation of antibiotics, while effective, risks generating transformation products (TPs) that may retain or amplify ecological threats, including antibiotic resistance gene (ARG) induction. This study addresses this challenge by developing a predictive, density functional theory (DFT)-informed machine learning (ML) framework that maps complex degradation networks and quantitatively assesses associated environmental risks. Using tetracycline as a model compound, we constructed an unprecedented reaction network of over 120 steps and 9,533 elementary reactions. To overcome the computational bottleneck of evaluating this network, an ML model was trained on DFT data to rapidly predict Gibbs free energy changes, enabling the first large-scale, network-wide thermodynamic feasibility assessment. This hybrid DFT-ML approach was successfully validated against the complex pathways of ciprofloxacin. Furthermore, a systematic pathway scoring system was developed to rank routes by integrating Diversity, Ecotoxicity, Biodegradability, and Feasibility of pathway (DEBF). Crucially, our analysis reveals that several TPs, particularly hydroxylated, amide ketone, and aminated species, pose a heightened risk of promoting antibiotic resistance. DFT-based docking studies confirm that these TPs form strong hydrogen bonds with key residues (Ile339A, Lys337A, Asp363A) in the tetM protein, potentially enhancing its binding affinity and ARG induction. This work provides a transformative strategy that bridges molecular-level reaction energetics with ecological risk assessment, guiding the development of photocatalytic water treatment processes that are not only efficient but also mitigate the unintended amplification of antimicrobial resistance.</p></details> | Theoretical and Computational Chemistry; Catalysis; Computational Chemistry and Modeling; Machine Learning; Photocatalysis | <details><summary>Show</summary><p>[{"title": "SI of Mapping Antibiotic Photocatalytic Transformation and Resistance Risks with a DFT-Informed Machine Learning Workflow", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b55078a94eede154dbac0f/original/si-of-mapping-antibiotic-photocatalytic-transformation-and-resistance-risks-with-a-dft-informed-machine-learning-workflow.pdf"}]</p></details> |
| **[Theory Meets Experiment in Ammonia Decomposition on Li14Cr2N8O: From Order to Disorder Under Reaction Conditions](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b46f1923be8e43d6237d65/original/theory-meets-experiment-in-ammonia-decomposition-on-li14cr2n8o-from-order-to-disorder-under-reaction-conditions.pdf)** | 2025-09-03 | 10.26434/chemrxiv-2025-1fx3j | <details><summary>Show</summary><p>Heterogeneous catalysts have long been considered rigid structures hosting localized active sites, but growing evidence from both experiments and simulations is revealing a more dynamic picture in which the entire catalyst evolves under reaction conditions. In this study, we explore such behavior in Li14Cr2N8O, a lithium chromium nitride oxide recently proposed as a candidate for ammonia decomposition. Using machine learning-accelerated molecular dynamics, combined with in situ X-ray diffraction and catalytic activity measurements, we show that the pristine material undergoes significant structural transformation upon exposure to ammonia at elevated temperature. Surface disorder, lithium mobility, and the progressive formation of amides and imides give rise to a reactive interface, where chromium centers mediate key redox processes. These interfacial fluctuations create the conditions necessary for key steps in ammonia decomposition, including N–N coupling, hydride formation and hydrogen release. Our findings highlight the importance of a global, dynamic view of hetereogenous catalysts under operando conditions, where activity arises not from predefined sites but from the evolving nature of the catalyst.</p></details> | Theoretical and Computational Chemistry; Catalysis; Theory - Computational; Machine Learning; Heterogeneous Catalysis | <details><summary>Show</summary><p>[{"title": "Supplementary Information ", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b4723f728bf9025e706669/original/supplementary-information.pdf"}]</p></details> |
| **[Optimizing Drug Activity Using Docking-Informed Machine Learning](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b6f727a94eede1541ec4da/original/optimizing-drug-activity-using-docking-informed-machine-learning.pdf)** | 2025-09-02 | 10.26434/chemrxiv-2025-vg35z-v2 | <details><summary>Show</summary><p>Bayesian optimization (BO) has been widely used to optimize drug potency towards a protein target by searching through fixed libraries of molecules and selecting compounds guided by machine learning (ML) predicted activities. We find that by pre-computing docking scores for the full library, and by using estimated binding energies and 3D descriptors from docking as features in the ML model, BO can be significantly accelerated relative to using traditional 2D features such as molecular fingerprints. Additionally, we have observed that a docking-based initialization scheme is often superior to the commonly-used diversity-based or random initialization. Using docking-based features and docking-based initialization required on average 24% (up to 77%) fewer data points to find the most active compound and gave on average 32% (up to 159%) improvement in enrichment factors, relative to a typical BO approach. We applied our method to 14 ChEMBL data sets, and 4 more highly challenging LIT-PCBA data sets with low hit rates and high molecular diversity. Our approach combines the generality of structure-based virtual screening (SBVS) with the inference power of ML ligand-based virtual screening (LBVS) to offer a more data-efficient hybrid approach.</p></details> | Theoretical and Computational Chemistry; Biological and Medicinal Chemistry; Drug Discovery and Drug Delivery Systems; Machine Learning; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supplementary Information: Optimizing Drug Activity Using Docking-Informed Machine Learning", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b6f727a94eede1541ec4e5/original/supplementary-information-optimizing-drug-activity-using-docking-informed-machine-learning.pdf"}]</p></details> |
| **[A general small language model (SLM) approach to examining scientific trends through conference proceedings: application to the 2019 and 2024 annual meetings of the Brazilian Chemical Society ](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b5c50723be8e43d653eae1/original/a-general-small-language-model-slm-approach-to-examining-scientific-trends-through-conference-proceedings-application-to-the-2019-and-2024-annual-meetings-of-the-brazilian-chemical-society.pdf)** | 2025-09-02 | 10.26434/chemrxiv-2025-vjqhg-v2 | <details><summary>Show</summary><p>Large Language Models (LLMs) are a machine learning technique that has transformed natural language processing. However, their large computational demands limit their accessibility, leading to the development of Small Language Models (SLMs), which, by running locally on a microcomputer, made AI-driven language processing and enhanced control for text analysis widely accessible. In this work, we use an SLM to analyze the evolution of Chemistry in Brazil by comparing data from the 2019 and 2024 Brazilian Chemical Society meetings (RASBQ). We demonstrate the viability of SLMs for extracting and structuring large volumes of text from scientific events collected in books of abstracts, thus enabling comprehensive comparative analyses that would otherwise be impractical. Our methodology extracts abstracts from the RASBQ digital proceedings and processes them using SLMs. These models converted the textual content into structured, manipulable data, thus enabling us to conduct a semantic and statistical analysis of the two events. The results highlight how SLMs can efficiently transform unstructured scientific proceedings into tractable data, thereby saving significant time and resources. The comparison between the 2019 and 2024 events revealed notable changes in thematic distribution, institutional participation, and potential regional impacts, underscoring the importance of data standardization in automated analyses. A Portuguese version of the Results and Discussion section is presented in the Supplementary Information.</p></details> | Theoretical and Computational Chemistry; Chemical Education; Machine Learning | <details><summary>Show</summary><p>[{"title": "Supplementary Material", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b5c57d23be8e43d653f49d/original/supplementary-material.pdf"}]</p></details> |
| **[Materials properties predictions: the role of numerical tokens in Large Language Models.](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68ae855923be8e43d63892f6/original/materials-properties-predictions-the-role-of-numerical-tokens-in-large-language-models.pdf)** | 2025-09-02 | 10.26434/chemrxiv-2025-ln8gv | <details><summary>Show</summary><p>Developing computational approaches to accurately predict materials properties has the potential to greatly accelerate materials research by enabling large-scale screening. However, for crystal materials, capturing their 3D structure and their periodicity is critical to make accurate properties predictions. While traditional deep learning approaches have showed significant improvements, they lack the ability to properly convey structure, periodicity and complex crystallographic information. Conversely, since crystal text descriptions can be very expressive, LLMs-based properties prediction models are promising. Here, I finetuned a state- of-the-art LLM-based model developed by Rubungo et al. [1] that predicts crystal bandgaps based on crystal text descriptions and used it as my baseline. I then developed an architecture that explicitly leverages numerical tokens in the text descriptions. I showed a 15meV improvement in the test error (MAE) over my baseline, more than the gain showcased by Rubungo et al. over other state-of-the- art models [1]. I also investigated newer embedding models to test their capabilities to extract information from text descriptions with numerical information. The best of these embeddings-based architectures, obtained with the E5-large model, yielded results comparable to my baseline, without the need to finetune the LLM embedding part. These newly developed architectures offer ways to push the limits of materials properties prediction.</p></details> | Theoretical and Computational Chemistry; Materials Science; Machine Learning; Artificial Intelligence; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[]</p></details> |
| **[Efficient Decoy Selection to Improve Virtual Screening Using Machine Learning Models](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b470ce728bf9025e7040b0/original/efficient-decoy-selection-to-improve-virtual-screening-using-machine-learning-models.pdf)** | 2025-09-01 | 10.26434/chemrxiv-2025-2hh8z-v2 | <details><summary>Show</summary><p>Machine learning models using protein-ligand interaction fingerprints show promise as target-specific scoring functions in drug discovery, but their performance critically depends on the underlying decoy selection strategies. Recognizing this critical role in model performance, various decoy selection strategies were analyzed to enhance machine learning models based on the Protein per Atom Score Contributions Derived Interaction Fingerprint (PADIF). We explored three distinct workflows for decoy selection: (1) random selection from extensive databases like ZINC15, (2) leveraging recurrent non-binders from highthroughput screening (HTS) assays stored as dark chemical matter, and (3) data augmentation by utilizing diverse conformations from docking results. Active molecules from ChEMBL, combined with these decoy approaches, were used to train and test different machine learning models based on PADIF. The final validation was done by confirming experimentally determined inactive compounds from the LIT-PCBA dataset. Our findings reveal that models trained with random selections from ZINC15 and compounds from dark chemical matter closely mimic the performance of those trained with actual non-binders, presenting viable alternatives for creating accurate models in the absence of specific inactivity data. Furthermore, all models showed an enhanced ability to explore new chemical spaces for their specific target and enhanced the top active compound selection over classical scoring functions, thereby boosting the screening power in molecular docking. These findings demonstrate that appropriate decoy selection strategies can maintain model accuracy while expanding applicability to targets even when lacking extensive experimental data.</p></details> | Theoretical and Computational Chemistry; Biological and Medicinal Chemistry; Machine Learning; Artificial Intelligence; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supporting Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b4713ea94eede154bd56dc/original/supporting-information.pdf"}]</p></details> |
| **[Can crystalline polymorphism be predicted from single-molecule properties?](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b40ae1728bf9025e66d753/original/can-crystalline-polymorphism-be-predicted-from-single-molecule-properties.pdf)** | 2025-09-01 | 10.26434/chemrxiv-2025-n71h3-v2 | <details><summary>Show</summary><p>Understanding the crystallization of molecules, and specifically the appearance of polymorphs, is a great challenge to modern chemistry, with both fundamental and practical aspects. Here, motivated by the established ability of Machine Learning (ML) algorithms to perform classification tasks, we harness ML-based tools and existing chemical datasets to ask whether the existence of polymorphs of a molecular crystal can be predicted based solely on properties of the corresponding single molecules. We implemented an algorithm that can predict the existence of polymorphism with an average accuracy of 65\%. Although this is not enough to generate a reliable “polymorph predicting” engine, our results reveal intriguing trends. We suggest that the limited performance of the model reflects the inherent bias of crystallographic data towards monomorphs. Namely, the fact that only one crystal form of a certain compound has been observed in experiments to date does not rule out the possibility that additional stable crystal structures could exist. Our results provide statistical support for a broadly held claim in the crystal engineering community that the proportion of possible polymorphs is, in fact, much larger than represented in the data.</p></details> | Theoretical and Computational Chemistry; Physical Chemistry; Organic Chemistry; Machine Learning; Quantum Mechanics; Crystallography – Organic | <details><summary>Show</summary><p>[{"title": "Supporting information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b40ae1728bf9025e66d75e/original/supporting-information.pdf"}]</p></details> |
| **[Enhancing Electrostatic Embedding for ML/MM Free Energy Calculations](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b34499728bf9025e4f62d4/original/enhancing-electrostatic-embedding-for-ml-mm-free-energy-calculations.pdf)** | 2025-09-01 | 10.26434/chemrxiv-2025-ktjh4-v3 | <details><summary>Show</summary><p>Hybrid ML/MM approaches that combine machine learning (ML) potentials with molecular mechanics (MM) potentials offer a promising balance between computational cost and accuracy. Most ML/MM simulations reported to date employ mechanical embedding schemes, and rely on Lennard-Jones and Coulomb potentials to model intermolecular interactions between the ML and MM regions. A promising approach to improving ML/MM schemes is to use electrostatic embedding, where polarization effects on the ML region by the MM region are explicitly incorporated. The electrostatic machine learning embedding (EMLE) method has been developed for this purpose.  Here, we compute absolute hydration free energies for a set of small organic molecules to derive robust methodologies for training EMLE models using quantum mechanical data. We establish protocols for fine-tuning the static and induced components of electrostatic interactions and evaluate the accuracy limits of fitting these components to first-principles calculations. We also introduce an empirical adjustment to enhance agreement with experimental results, strengthening the competitiveness of ML/MM simulations relative to state-of-the-art methods. Overall, our findings provide valuable insights into the challenges and opportunities of electrostatic embedding ML/MM simulations, and offer strategies for achieving robust modelling of classes of drug-like molecules where the accuracy of conventional MM force fields fall short.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Machine Learning; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supporting Information: Enhancing Electrostatic Embedding for ML/MM Free Energy Calculations", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b34499728bf9025e4f62e1/original/supporting-information-enhancing-electrostatic-embedding-for-ml-mm-free-energy-calculations.pdf"}]</p></details> |
| **[An excitation strategy for the initial condition generation for surface hopping trajectories using electron-only dynamics including explicit laser pulses](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b178c023be8e43d6c6e927/original/an-excitation-strategy-for-the-initial-condition-generation-for-surface-hopping-trajectories-using-electron-only-dynamics-including-explicit-laser-pulses.pdf)** | 2025-09-04 | 10.26434/chemrxiv-2025-3p8c0 | <details><summary>Show</summary><p>Nonadiabatic dynamics simulations, e.g., via trajectory surface hopping, are nowadays used regularly to describe various photoinduced phenomena in molecules. For a number of reasons, in the setup of such simulations, the actual photoexcitation process is often described by rather crude and approximative excitation schemes (e.g., vertical excitation by a delta pulse), and more attention is directed to the ensuing dynamics simulations. However, several studies have implied the importance of properly considering the spatial, temporal, and spectral details of the exciting laser pulse. Here, we suggest the “electron-only explicit” (“EOE”) excitation scheme for setting up trajectory surface hopping initial conditions based on an explicit laser pulse at little computational cost. The scheme is based on solving the time-dependent electronic Schrödinger equation including the explicit influence of a laser pulse within the frozen-nuclei approximation. The obtained time-dependent electronic populations are then used to stochastically select the initial electronic states in a post-processing step. Here, the electronic populations are renormalized such that one excites a reasonable fraction of the initial condition even when operating well within the weak-field regime. We illustrate the new excitation scheme, which is available as part of the SHARC dynamics package, by means of several simulation sets of protonated formaldimine (CH2NH2+) excited by different laser pulses.</p></details> | Theoretical and Computational Chemistry; Physical Chemistry; Theory - Computational; Photochemistry (Physical Chem.); Spectroscopy (Physical Chem.) | <details><summary>Show</summary><p>[{"title": "Supporting Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b18913a94eede15462959e/original/supporting-information.pdf"}]</p></details> |
| **[Unified Mechanics on Thermodynamics, Classical Mechanics, and Quantum Mechanics](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b025b823be8e43d67d158a/original/unified-mechanics-on-thermodynamics-classical-mechanics-and-quantum-mechanics.pdf)** | 2025-09-03 | 10.26434/chemrxiv-2024-35kds-v7 | <details><summary>Show</summary><p>Classical and quantum mechanics laws are rebuilt in the framework of new thermodynamics: Electrons condense to form the hot protons and cool protons (neutrons), and then construct the nucleus by the thermal difference force. An atom is an adiabatic electron-gas system. Heat is the sum of variations in kinetic energy, system potential, and environmental potential within a system. Force is the heat gradient over distance in the direction of transfer. Two kinds of forces exist between two particles, resulting from the relative thermal difference and the relative difference in volumetric density. Mutual repulsion and attraction simultaneously exist. Interaction doesn’t need a medium. As achievements, a series of concise equations is derived that incorporates spatial parameters, mass, and temperature. Electricity is a nonessential concept. 1) The equilibrium distance of molecular interaction: L_e=∛((3π^(α-1) m_A g)/(4N_A kT)); 2) the atom structure is built based on the radii of atoms and the temperature of electrons. An atom has only one isothermal orbital. The other are non-isothermal orbits constituting a tetrahedron. Each orbit accommodates 2 electrons with opposite spins; 3) the 1st ionization energies (IE) are close to the kinetic energies of surface electrons, 3/2 kT_0; 4) the strengths of chemical bonds and H-bonds are calculated, e.g., the strength of O-O: ∆E=2∙3k/2(2/√3-1)T_0; 5) Light is the heat transfer from a hot electron to a cold electron. The rate of hot electrons determines the velocity, but the rotating rate of the hot molecule determines the frequency. For example, a H2 on Earth transfers heat to a cold particle with velocity, v_exo=9.59×〖10〗^7 sin(πf_e t)cos⁡(πf_e t)cos⁡(2πf_p t). Hence, the energy transfers by piece and piece. It is the so-called wave-particle duality.</p></details> | Theoretical and Computational Chemistry; Physical Chemistry; Earth, Space, and Environmental Chemistry; Computational Chemistry and Modeling; Theory - Computational; Spectroscopy (Physical Chem.) | <details><summary>Show</summary><p>[]</p></details> |
| **[Excited states of peridinin in PCP: Polarizable embedding QM/MM study](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b70c6e728bf9025ed4cf3d/original/excited-states-of-peridinin-in-pcp-polarizable-embedding-qm-mm-study.pdf)** | 2025-09-03 | 10.26434/chemrxiv-2025-bwnnj-v2 | <details><summary>Show</summary><p>A complete mechanistic understanding of the excitation energy transfer in the Peridinin Chlorophyll Protein (PCP) complex has been a long sought after goal due to the protein's high light-harvesting efficacy. Multiple factors including the relative energies, electronic couplings, and the chromophores' interactions with the protein environment collectively contribute to the overall efficiency of the energy transfer in the PCP complex. Here we focus specifically on the energy ordering of the bright excited states of the eight peridinin chromophores in monomeric PCP. We provide a detailed analysis of the effects of the electrostatic interactions of peridinins with the protein matrix on excitation energies. Our results point to peridinins 611 and 621 as the lowest-energy excitation sites among the eight carotenoid chromophores. Despite the substantial transition dipole moment, the effects of the environment polarization on the computed excitation energy are found to be minor. This conclusion is supported by both the zero-order and linear-response TDDFT/EFP polarizable embedding QM/MM calculations. Our simulations also indicate that methods beyond molecular dynamics with conventional molecular mechanics forcefields should be used to sample the configurational space of the peridinins. </p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Theory - Computational | <details><summary>Show</summary><p>[{"title": "Supplementary Information ", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b70c6f728bf9025ed4cf46/original/supplementary-information.pdf"}]</p></details> |
| **[Transient Microkinetic Modeling of Electrochemical Reactions: Capturing Unsteady Dynamics of CO Reduction and Oxygen Evolution](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b6d526728bf9025ecc484d/original/transient-microkinetic-modeling-of-electrochemical-reactions-capturing-unsteady-dynamics-of-co-reduction-and-oxygen-evolution.pdf)** | 2025-09-03 | 10.26434/chemrxiv-2025-rggk3-v2 | <details><summary>Show</summary><p>Electrochemical processes, such as water splitting and carbon dioxide/monoxide (CO(2)) reduction, will play a prominent role in the ongoing quest for mitigating climate change. For such reactions, microkinetic modeling (MKM) is a valuable tool to relate electrolyzer operating conditions, such as pH, temperature, and pressure, to current densities and faradaic efficiencies. However, previous studies have solely focused on steady-state modeling of electrochemical kinetics. Here, we perform unsteady-state MKM (USS-MKM) with and without potential sweeping to capture transient dynamics and realistically model reaction kinetics. Our analysis demonstrates that sweeping leads to accurate description of the dynamics of current-potential relationships that arise during experimental linear sweep voltammetry or staircase voltammetry measurements. The proposed approach is validated using CO reduction and oxygen evolution reactions, where reasonable agreement is observed between our long-time USS-MKM results and previously reported steady-state MKM data. We also show that our approach leads to reasonable agreement with experimental CO reduction current density data. Moreover, our proposed approach is automated, scaling to large reaction mechanisms, and enables a graphical representation of electrochemical reaction networks. Overall, by enabling USS-MKM with potential sweeping, our framework simplifies the study of complex electrocatalytic mechanisms and offers valuable insights into their operation under dynamic conditions. </p></details> | Theoretical and Computational Chemistry; Catalysis; Chemical Engineering and Industrial Chemistry; Theory - Computational; Electrocatalysis; Heterogeneous Catalysis | <details><summary>Show</summary><p>[{"title": "Supporting Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b6d526728bf9025ecc4858/original/supporting-information.pdf"}]</p></details> |
| **[Enhanced Sampling of Ligand Binding Coupled to RNA Conformational Dynamics](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b60462728bf9025ea86c82/original/enhanced-sampling-of-ligand-binding-coupled-to-rna-conformational-dynamics.pdf)** | 2025-09-03 | 10.26434/chemrxiv-2025-hnc4c-v2 | <details><summary>Show</summary><p>Noncoding RNAs have recently emerged as promising targets for therapeutic drugs. However, their high conformational flexibility presents significant challenges in structure-based rational drug discovery. Therefore, it is essential to gain detailed mechanistic insights into the millisecond-timescale ligand recognition process coupled to the conformational dynamics of RNA. Enhanced sampling simulations can help investigate these intricate dynamical processes by effectively exploring the rugged conformational landscapes of RNA-ligand complexes. In this study, we applied the On-the-fly Probability Enhanced Sampling (OPES) algorithm and its flooding variant to compute the free energy surface, kinetics, and binding mechanism for three different natural and synthetic ligands to a prototypical RNA aptamer. Our simulations revealed the dynamic coupling between ligand recognition and the base-flipping transition of a cytosine (C27) nucleotide. In addition to reproducing the relative trends of binding affinities and residence times, our results uncovered a competing effect between the conformational selection and the induced fit mechanisms. The preference for one mechanism over the other is influenced by the binding affinity. By integrating these mechanistic insights with the binding affinity and kinetics derived from our simulations, we provide an atomistically detailed understanding of the RNA-ligand recognition process.</p></details> | Theoretical and Computational Chemistry; Physical Chemistry; Computational Chemistry and Modeling; Theory - Computational; Biophysical Chemistry | <details><summary>Show</summary><p>[{"title": "Supporting Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b60462728bf9025ea86c95/original/supporting-information.pdf"}]</p></details> |
| **[CRISP: Enhancing ASE Workflows with Advanced Molecular Simulation Post-Processing](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b17ed5728bf9025e145d7d/original/crisp-enhancing-ase-workflows-with-advanced-molecular-simulation-post-processing.pdf)** | 2025-09-02 | 10.26434/chemrxiv-2025-dht8g | <details><summary>Show</summary><p>Molecular simulations are invaluable for analysing molecular systems, but existing post-processing tools are often limited by a lack of customisation, interactivity, and efficiency with large datasets. To address this, we developed CRISP (Comprehensive Repository for Insightful Simulation Post-Processing), an open-source Python toolkit designed to enhance workflows within the Atomic Simulation Environment (ASE). CRISP provides a versatile platform for detailed analysis and visualisation, featuring a customizable and modular design, various static analysis methods, interactive 3D visualisations, and parallel processing capabilities optimised for high-performance computing. We demonstrate its effectiveness through case studies, including the analysis of statistical convergence in zeolite simulations, subsampling large datasets for machine learning, and analysing the dynamic stability of atomic clusters. CRISP effectively bridges the gap between raw simulation data and actionable insights, offering an efficient solution for researchers and saving significant time in code development. </p></details> | Theoretical and Computational Chemistry; Theory - Computational; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supplementary Information of, CRISP: Enhancing ASE Workflows with Advanced Molecular Simulation Post-Processing", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b1831ea94eede15461c165/original/supplementary-information-of-crisp-enhancing-ase-workflows-with-advanced-molecular-simulation-post-processing.pdf"}]</p></details> |
| **[PyGraSO: Analytical Nuclear Derivatives of Spin-Orbit Coupling for Intersystem Crossing Dynamics Simulations](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b10d06a94eede1544eda8d/original/py-gra-so-analytical-nuclear-derivatives-of-spin-orbit-coupling-for-intersystem-crossing-dynamics-simulations.pdf)** | 2025-09-01 | 10.26434/chemrxiv-2025-0ssh5 | <details><summary>Show</summary><p>We present PyGraSO, an open-source program for the efficient and accurate calculation of spin-orbit coupling matrix elements (SOCMEs) and their analytical nuclear derivatives within the framework of time-dependent density functional theory. The SOCME derivatives are crucial physical quantities for capturing spin-vibronic coupling (SVC) effects associated with the induced SOC values via molecular vibrations. These SVC effects are often crucial for describing spin-forbidden or spin-flip processes, such as intersystem crossing (ISC), reverse ISC (RISC), and phosphorescence.   Previously, these derivatives could only be obtained through numerical differentiation. With our new implementation, they can now be computed based on analytical formulas, underpinning a significant advance toward more accurate and efficient dynamics simulations of spin-forbidden processes where SVC effects are vital.   Although developing the analytical nuclear derivative method is challenging, our implementation achieves substantial efficiency gains over numerical approaches, yielding up to a 36-fold speedup in benchmark tests. To showcase its applicability, we performed excited-state dynamics simulations based on thermal vibration correlation function (TVCF) formalism, incorporating SVC effects via SOCME derivatives, to predict RISC rate constants ($k_\mathrm{RISC}$) for a large set of 126 thermally activated delayed fluorescence (TADF) molecules. The results demonstrated that the inclusion of SVC effects improves agreement with experimental data, particularly for molecules with small reorganization energies, where the other methods that do not incorporate SOCME derivatives tend to fail.</p></details> | Theoretical and Computational Chemistry; Theory - Computational | <details><summary>Show</summary><p>[{"title": "Supplementary information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b1117123be8e43d6b5eb2d/original/supplementary-information.pdf"}]</p></details> |
| **[Semiempirical Quantum Chemistry in the Age of ab initio Data and Differentiable Programming: I. Differentiable Molecular Orbital Theory](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b45169728bf9025e6d3cfb/original/semiempirical-quantum-chemistry-in-the-age-of-ab-initio-data-and-differentiable-programming-i-differentiable-molecular-orbital-theory.pdf)** | 2025-09-01 | 10.26434/chemrxiv-2025-k1zg8-v2 | <details><summary>Show</summary><p>Semiempirical quantum chemistry (SQC) methods offer fast quantum chemical insights by constructing and solving a parametric effective minimal basis Fock matrix. Establishing suitable parameterizations has long been a challenging and time-consuming task involving tedious grid searches or costly finite-difference gradients of carefully crafted loss functions based on select experimental data. The growing availability of differentiable programming environments that leverage algorithmic differentiation to obtain complex derivatives together with access to a wealth of reliable reference data from ab initio calculations offers a new and more efficient approach. In this work, we extend a previous, basic implementation of SQC methods in PyTorch [J. Chem. Theory Comput. 16, 4951-4962 (2020)] by including global algorithmic considerations into the code design. This allows for improved general applicability and establishes a robust back-end for rapid SQC parameterizations. In particular, we address the general differentiability of the eigensolver and the iterative SCF procedure. The new implementation offers drastic improvements in computing costs as well as memory footprint and provides increased stability in gradient evaluation. We highlight the importance of these advances and their improvements over existing formulations and illustrate their role in the context of SQC parameterization.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Theory - Computational | <details><summary>Show</summary><p>[{"title": "Supplementary Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68b45169728bf9025e6d3d04/original/supplementary-information.pdf"}]</p></details> |

