# Weekly Papers
This project automatically fetches the latest papers from ChemRxiv based on categories.

Only the most recent articles for each category are retained, up to a maximum of 100 papers.

You can click the "Watch" button to receive daily email notifications.

Last update: 2025-06-09

| **Title** | **Date** | **Doi** | **Abstract** | **Categories** | **Suppitems** |
| --- | --- | --- | --- | --- | --- |
| **[Optimization of Generator Reward Functions Targeting Non-covalent KRAS Inhibitors](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/684004293ba0887c339f7032/original/optimization-of-generator-reward-functions-targeting-non-covalent-kras-inhibitors.pdf)** | 2025-06-06 | 10.26434/chemrxiv-2025-2lw7f-v2 | <details><summary>Show</summary><p>While machine learning (ML) and artificial intelligence (AI) technologies for molecular generation have seen rapid progress in the last five years, there is still a need for careful set up and optimization of these tools. This study documents the optimization of reward functions for a reinforecement learning (RL)-based generator targeting non-covalent inhibitors of KRAS. In particular, we study the relative effect of additive versus multiplicative reward components in a multiobjective function using docking environments and pharmaocphore modeling. Our findings illustrate how to maximize the outcome of a target criterion, using the number of molecules that meet or surpass a target docking score. In the process, we find an apparent compounding effect of docking and pharmacophore modeling on the generators’ results. From these results, we identify and assess a promising non-covalent inhibitor candidate molecule based on docking score and likely interactions across a variety of KRAS mutants.</p></details> | Theoretical and Computational Chemistry; Biological and Medicinal Chemistry; Drug Discovery and Drug Delivery Systems; Artificial Intelligence; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Figure S1", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/684004293ba0887c339f703c/original/figure-s1.pdf"}, {"title": "Generated molecules for RI-NO-MULT", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/684004293ba0887c339f7041/original/generated-molecules-for-ri-no-mult.csv"}, {"title": "Generated molecules for RI-MULT-ADD", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/684004293ba0887c339f7046/original/generated-molecules-for-ri-mult-add.csv"}, {"title": "Generated molecules for RI-MULT-NO", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/684004293ba0887c339f704b/original/generated-molecules-for-ri-mult-no.csv"}, {"title": "Generated molecules for RI-MULT-MULT-1", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/684004293ba0887c339f7050/original/generated-molecules-for-ri-mult-mult-1.csv"}, {"title": "Generated molecules for RI-MULT-MULT-2", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/684004293ba0887c339f7055/original/generated-molecules-for-ri-mult-mult-2.csv"}, {"title": "Generated molecules for RI-MULT-MULT-3", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/684004293ba0887c339f705a/original/generated-molecules-for-ri-mult-mult-3.csv"}, {"title": "Generated molecules for RI-NO-NO", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/684004293ba0887c339f705f/original/generated-molecules-for-ri-no-no.csv"}]</p></details> |
| **[TCM-Navigator, a Deep Learning-based Workflow for Generation and Evaluation of Traditional Chinese Medicine-like Compounds for Drug Development](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683ea988c1cb1ecda0e61d90/original/tcm-navigator-a-deep-learning-based-workflow-for-generation-and-evaluation-of-traditional-chinese-medicine-like-compounds-for-drug-development.pdf)** | 2025-06-06 | 10.26434/chemrxiv-2025-xrcfk | <details><summary>Show</summary><p>Traditional Chinese Medicine (TCM) has long been regarded as a valuable resource for modern drug discovery. However, the limited availability of recorded entities and information, the complexity and sparsity of the herb–ingredient–target–disease network, and inconsistencies in data representation hinder the effectiveness of high-throughput screening approaches. While some therapeutically valuable compounds from TCM have been discovered through manual experimental screening, such methods are time-consuming and require substantial human resources. To address these challenges, we developed a data-driven and deep learning–based workflow, TCM-Navigator, that enables the in-silico generation, quality control, and physics-based evaluation of TCM-like molecules. The generation is done by TCM-Generator, a transfer learning- and LSTM-based chemical language model that generates standardized, hierarchically structured, and high-throughput–friendly datasets of TCM-like molecules. In this study, we generated a target-nonspecific dataset comprising 3.7 million TCM-like molecules, expanding the number of entities in existing TCM datasets by more than 100-fold. The workflow also enables flexible, goal-driven molecule generation customized for specific targets, yielding three target-specific datasets and multiple high-potential target-ligand pairs. The quality control is done by TCM-Identifier, the first quantitative model specifically designed to capture unique characteristics of TCM, using an AttentiveFP framework with Message Passing Neural Networks (MPNNs). TCM-Identifier is expected to serve as an essential evaluation and guidance tool for TCM-related drug development. Our workflow bridges cutting-edge data science—including deep learning—with biomedical research to tackle longstanding challenges in target identification and molecular design. Its adaptable framework is also transferable to interdisciplinary innovation beyond drug development. 
</p></details> | Theoretical and Computational Chemistry; Biological and Medicinal Chemistry; Bioinformatics and Computational Biology; Computational Chemistry and Modeling; Artificial Intelligence | <details><summary>Show</summary><p>[{"title": "Supplementary figures", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683eb0e93ba0887c3366623d/original/supplementary-figures.pdf"}, {"title": "Supplementary Methods ", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683eb1d91a8f9bdab562bfed/original/supplementary-methods.pdf"}]</p></details> |
| **[Co-Intelligent Design of Catalysis Research with Large Language Models: Hype or Reality?](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683dd510c1cb1ecda0ca5ea0/original/co-intelligent-design-of-catalysis-research-with-large-language-models-hype-or-reality.pdf)** | 2025-06-05 | 10.26434/chemrxiv-2025-5d76r-v2 | <details><summary>Show</summary><p>Large language models are making a strong impact on broad fields of science, pushing the efforts to minimize hallucinations and to support  specialized language. Co-intelligence, the synergy between human and artificial intelligences, can have a strong potential in research through the use of large language models. This Viewpoint focused on the question: Can large language models collaborate with chemists in the design of catalysis research? In a conversation based on a few prompts giving minimal guidance, ChatGPT-4o proved its co-design abilities at different levels of detail, making a major contribution to the definition of a research project concept and its workflows. Relevant flaws were also observed, highlighting the importance of involving critical thinking from highly-educated researchers. The assessment of the results by ChatGPT-4o itself and other large language models, including DeepSeek and Gemini, was also explored.</p></details> | Theoretical and Computational Chemistry; Catalysis; Machine Learning; Artificial Intelligence; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supporting Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683dd511c1cb1ecda0ca5ebc/original/supporting-information.pdf"}]</p></details> |
| **[Benchmarking ML in ADMET predictions: the practical impact of feature representations in ligand-based models](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683d7c251a8f9bdab53dd9e1/original/benchmarking-ml-in-admet-predictions-the-practical-impact-of-feature-representations-in-ligand-based-models.pdf)** | 2025-06-05 | 10.26434/chemrxiv-2023-x6tjr-v7 | <details><summary>Show</summary><p>This study, focusing on predicting Absorption, Distribution, Metabolism, Excretion, and Toxicology (ADMET) properties, addresses the key challenges of ML models trained using ligand-based representations. We propose a structured approach to data feature selection, taking a step beyond the conventional practice of combining different representations without systematic reasoning. Additionally, we enhance model evaluation methods by integrating cross-validation with  statistical hypothesis testing, adding a layer of reliability to the model assessments. Our final evaluations include a practical scenario, where models trained on one source of data are evaluated on a different one. This approach aims to bolster the reliability of ADMET predictions, providing more dependable and informative model evaluations.</p></details> | Theoretical and Computational Chemistry; Machine Learning; Artificial Intelligence; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[]</p></details> |
| **[POLARIS: Perovskite Optimization using LLM-Assisted Refinement and Intelligent Screening](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683c77ef1a8f9bdab524557c/original/polaris-perovskite-optimization-using-llm-assisted-refinement-and-intelligent-screening.pdf)** | 2025-06-04 | 10.26434/chemrxiv-2025-qqchd | <details><summary>Show</summary><p>We present a comprehensive and reproducible pipeline that unites literature mining, molecular graph generation, and uncertainty-aware predictive modeling to accelerate the design of organic spacer cations for 2D halide perovskites (HPs). Despite the critical influence of spacer chemistry on phase stability, excitonic behavior, transport properties and environmental robustness, the design space of HPs remains underexplored due to inconsistent reporting and limited structured datasets. To overcome this, we curated a diverse set of 200 experimental papers from various publishers and research groups into Google’s NotebookLM powered by Gemini, utilizing its retrieval-augmented generation (RAG) framework to extract synthesis-relevant metadata with high accuracy and reproducibility. To ensure data quality and consistency, we limited our selection to papers published in peer-reviewed journals with an impact factor above 10, focusing on studies with well-documented experimental protocols. Benchmarking against four other LLMs confirmed NotebookLM’s superior stability and minimal hallucination rate, making it ideal for hypothesis-driven data curation. From extracted IUPAC names, we constructed SMILES representations and augmented the dataset with over 10,000 ammonium-containing molecules from QM9. These were converted into graph-based molecular embeddings and used to train a multitask graph neural network coupled with a Gaussian process (GNN–GP) backend to predict optoelectronic and structural properties with uncertainty quantification. The Latent space clustering of the learned embeddings revealed chemically interpretable families of spacer candidates, which we cross-validated against ChatGPT-generated design heuristics. The convergence between unsupervised clustering and transformer-derived guidance highlights the power of combining LLMs with active learning to generate, test, and refine design hypotheses in underexplored chemical domains. This study demonstrates how fragmented literature can be transformed into actionable, structure–property insights through a tightly integrated informatics pipeline. Our approach lays the foundation for closed-loop, autonomous materials discovery and design and provides a scalable strategy for targeted development of next-generation HPs optoelectronics.</p></details> | Theoretical and Computational Chemistry; Materials Science; Hybrid Organic-Inorganic Materials; Machine Learning; Artificial Intelligence; Materials Chemistry | <details><summary>Show</summary><p>[{"title": "POLARIS: Perovskite Optimization using LLM-Assisted Refinement and Intelligent Screening", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683c7c551a8f9bdab524f1ea/original/polaris-perovskite-optimization-using-llm-assisted-refinement-and-intelligent-screening.pdf"}]</p></details> |
| **[Graph-based prediction of reaction barrier heights with on-the-fly prediction of transition states](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68395a543ba0887c33c60f6c/original/graph-based-prediction-of-reaction-barrier-heights-with-on-the-fly-prediction-of-transition-states.pdf)** | 2025-06-03 | 10.26434/chemrxiv-2025-w2kgt-v2 | <details><summary>Show</summary><p>The accurate prediction of reaction barrier heights is crucial for understanding chemical reactivity and guiding reaction design. Recent advances in machine learning (ML) models, particularly graph neural networks, have shown great promise in capturing complex chemical interactions. Here, directed message-passing neural networks (D-MPNNs) on graph overlays of the reactant and product graphs were shown to provide promising accuracies for reaction property prediction. They only rely on the change in molecular graphs as input and thus require no additional information during inference. However, the reaction barrier height intrinsically depends on the conformations of the reactants, transition state, and products, which are not taken into account in standard D-MPNNs. In this work, we present a hybrid approach where we combine the power of D-MPNNs predicting barrier heights with generative models predicting transition state geometries on-the-fly, only given the reaction graph. The resulting model thus only requires two-dimensional graph information as input, while internally leveraging three-dimensional information to increase accuracy. We furthermore evaluate the influence of additional physical features on D-MPNN models of reaction barrier heights, where we find that additional features only marginally enhance predictive accuracy and are especially helpful for small datasets. In contrast, our hybrid graph/coordinate approach reduces the error of barrier height predictions for all investigated datasets.</p></details> | Theoretical and Computational Chemistry; Machine Learning; Artificial Intelligence; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supplementary information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68395a543ba0887c33c60f78/original/supplementary-information.pdf"}]</p></details> |
| **[Hierarchical AF2RAVE for multi-conformation virtual screening targeting S100 Ca$^{2+}$-binding proteins](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6838bc83c1cb1ecda03218c1/original/hierarchical-af2rave-for-multi-conformation-virtual-screening-targeting-s100-ca-2-binding-proteins.pdf)** | 2025-06-03 | 10.26434/chemrxiv-2025-7j3gd | <details><summary>Show</summary><p>Protein function is driven by transitions between metastable conformations, many of which are not conserved across homologs, offering opportunities for selective drug design. Accurately modeling both backbone and sidechain metastability and generating structures suitable for rigid docking in high-throughput virtual screening, is thus desirable yet challenging. Here, we present a hierarchical AF2RAVE pipeline that integrates  AlphaFold2 with machine learning-based enhanced sampling to systematically explore the free energy landscape and metastability of protein systems, particularly at both backbone and sidechain levels. Applied to the calcium-binding S100 protein family, this approach enables the generation of diverse holo-like conformations, starting from sequence. Retrospective docking and enrichment testing with a new $Ca^{2+}$-S100B inhibitor dataset demonstrates that AF2RAVE-generated structures outperform standard AlphaFold2 and even outperform experimentally resolved X-ray structures in enrichment testing. Our results highlight the potential of AF2RAVE for high-throughput virtual screening and selective inhibitor discovery, particularly for challenging targets such as the $Ca^{2+}$-S100 family.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Artificial Intelligence | <details><summary>Show</summary><p>[{"title": "SI text and figures", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6838c0e83ba0887c33b24127/original/si-text-and-figures.pdf"}]</p></details> |
| **[Increasing trustworthiness of machine learning-based drug sensitivity prediction with a multivariate random forest approach](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6836eb70c1cb1ecda0f0e9a6/original/increasing-trustworthiness-of-machine-learning-based-drug-sensitivity-prediction-with-a-multivariate-random-forest-approach.pdf)** | 2025-06-02 | 10.26434/chemrxiv-2025-ml78s | <details><summary>Show</summary><p>Ensuring the trustworthiness of machine learning (ML) models in high-stake applications is crucial.  One such application is predicting anti-cancer drug sensitivity, where ML models are built with the final goal of integrating them into treatment recommendation systems for personalized medicine. Here, we propose a trustworthy multivariate random forest method, called MORGOTH. Besides standard regression and classification functions, MORGOTH allows for the simultaneous optimization of regression and classification tasks via a joint splitting criterion. Additionally, it provides a graph representation of the random forest to address model interpretability, and a cluster analysis of the leaves to measure the dissimilarity of new inputs from the training data to account for its reliability. While our approach is broadly applicable, we demonstrate its capabilities for anti-cancer drug sensitivity prediction by a comprehensive large-scale study on the Genomics of Drug Sensitivity in Cancer (GDSC) database. We trained single-drug as well as multi-drug models. In either case, MORGOTH clearly outperforms state-of-the-art neural network approaches. Moreover, we highlight an evaluation issue for multi-drug models and demonstrate that single-drug models consistently outperform them when evaluated fairly.</p></details> | Theoretical and Computational Chemistry; Machine Learning; Artificial Intelligence; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supplement", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6836ede73ba0887c337039bd/original/supplement.pdf"}]</p></details> |
| **[No Time for Nitrides: How Cobalt Alloying Promotes Iron Catalysts for Ammonia Decomposition](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683d894ec1cb1ecda0c33475/original/no-time-for-nitrides-how-cobalt-alloying-promotes-iron-catalysts-for-ammonia-decomposition.pdf)** | 2025-06-06 | 10.26434/chemrxiv-2025-nv4ht | <details><summary>Show</summary><p>The increasing demand for sustainable hydrogen production has driven interest in ammonia decomposition. Iron-based catalysts, widely used for ammonia synthesis, exhibit suboptimal performance in the reverse process due to their tendency to form iron nitrides. Alloying iron with cobalt has been reported to enhance catalytic activity, but the underlying microscopic mechanisms remain unclear. In this study, we employ machine learning-based molecular dynamics simulations and experimental validation to investigate the role of Fe-Co alloying in ammonia decomposition. Our simulations reveal that cobalt alloying into iron catalysts provides a dual promotional effect: it slightly lowers the free energy barrier for nitrogen recombination, which is the rate-determining step in ammonia decomposition, while significantly suppressing nitrogen migration into the bulk, thereby preventing nitride formation. These findings are corroborated by transient decomposition experiments and desorption measurements, which demonstrate enhanced activity and resistance to nitridation in FeCo alloys compared to monometallic iron catalysts. Our results provide fundamental atomistic insights into the synergistic effect of Fe-Co alloying, offering a pathway for the rational design of more efficient catalysts.</p></details> | Theoretical and Computational Chemistry; Catalysis; Theory - Computational; Machine Learning; Heterogeneous Catalysis | <details><summary>Show</summary><p>[]</p></details> |
| **[Automated Learning of GNN Ensembles for Predicting Redox Potentials with Uncertainty](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683e0487c1cb1ecda0ce5640/original/automated-learning-of-gnn-ensembles-for-predicting-redox-potentials-with-uncertainty.pdf)** | 2025-06-05 | 10.26434/chemrxiv-2025-0tq7j | <details><summary>Show</summary><p>Accurate prediction of redox potentials of iron (Fe) complexes, in tandem with uncertainty quantification, is essential to advance technologies related to electro-deposition and energy storage by enabling reliable modeling, guiding experimental design, and improving the efficiency of material discovery. Since experimental measurements and first-principles simulations are time-consuming, machine learning (ML) based on existing high-fidelity data  is a fast alternative. In this contribution, we present a general ML framework for automated learning of Graph Neural Network (GNN) ensembles for predicting redox potentials of Fe complexes and quantifying the corresponding uncertainty.  Using a recently reported high-fidelity dataset of transition metal complexes by Janet et al. [ACS Cent. Sci. 2020, 6, 4, 513–524], we leverage the DeepHyper hyperparameter optimization framework to optimize the hyperparameters of a user-defined GNN model to obtain a large set of high performing models in an automated way. This set of candidate models is then used to generate ensembles. We demonstrate that the proposed ensemble approach achieves superior performance in comparison to a single best-performing GNN model and also over a varied set of ML model classes applied to a standard benchmark.  Further, the proposed framework offers insights into the confidence of model predictions through epistemic uncertainty measures.  In fact, we demonstrate that through effective uncertainty quantification, our framework can be used to identify molecular structures and attributes that significantly affect model performance, particularly in  challenging datasets with complex ligand environments.  This general framework opens the door to a scalable and accurate high-throughput screening of transition metal complexes, facilitating the rapid discovery of new materials for energy and catalysis applications.</p></details> | Theoretical and Computational Chemistry; Inorganic Chemistry; Coordination Chemistry (Inorg.); Transition Metal Complexes (Inorg.); Machine Learning | <details><summary>Show</summary><p>[{"title": "Supporting Information: Predicting Redox Potentials of Iron Complexes with Ensemble Graph Neural Networks", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683e0966c1cb1ecda0cedb31/original/supporting-information-predicting-redox-potentials-of-iron-complexes-with-ensemble-graph-neural-networks.pdf"}]</p></details> |
| **[ConfRank+: Extending Conformer Ranking to Charged Molecules](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683d7c333ba0887c334258b2/original/conf-rank-extending-conformer-ranking-to-charged-molecules.pdf)** | 2025-06-05 | 10.26434/chemrxiv-2025-xkwk6 | <details><summary>Show</summary><p>We present a machine learning model for high-throughput energetic ranking of charged molecular conformers. Based on the ConfRank approach, the model is trained in a pairwise fashion to predict energy differences for pairs of conformers. By conditioning the model on dataset embedding vectors, we are able to train our model on two different reference levels simultaneously, allowing for a larger training dataset and to emulate multiple reference methods. In particular, we train our model on a large subset of the SPICE 2.0.1 dataset with references on the ωB97M-D3(BJ)/def2-TZVPPD range-separated hybrid meta-GGA DFT-level and a self-developed conformer dataset based on the GEOM dataset including r²SCAN-3c references. The result is a single multi-fidelity model that can reproduce both reference levels up to ML-typical model errors for small- and medium-sized molecules including the following elements: H, Li, B, C, N, O, F, Na, Mg, Si, P, S, Cl, K, Ca, Br, I. By including partial atomic charges obtained from the electronegativity equilibration charge model, our model incorporates information about the charge distribution in a molecule, allowing the treatment of charged closed-shell species and explicit treatment of electrostatic interactions. We test the ranking capability of the model on various datasets, paying special attention to molecular charges of -1, 0, 1. Throughout all tests, we find our model to be as accurate as current AIMNet2 and MACE-OFF23(L) models, while requiring an order of magnitude fewer parameters and matching the robustness of the state-of-the-art semi-empirical quantum method GFN2-xTB. </p></details> | Theoretical and Computational Chemistry; Organic Chemistry; Computational Chemistry and Modeling; Machine Learning; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supporting Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683d7f203ba0887c33428576/original/supporting-information.pdf"}]</p></details> |
| **[Using Deep Graph Neural Networks Improves Physics-Based Hydration Free Energy Predictions Even for Molecules Outside of the Training Set Distribution](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683cb1411a8f9bdab529bb1f/original/using-deep-graph-neural-networks-improves-physics-based-hydration-free-energy-predictions-even-for-molecules-outside-of-the-training-set-distribution.pdf)** | 2025-06-04 | 10.26434/chemrxiv-2025-b4b11 | <details><summary>Show</summary><p>The accuracy of computational models of water is key to atomistic simulations of biomolecules. Here we explore a decoupled framework that combines classical physics- based models with deep neural networks (DNNs) to correct residual error in hydration free energy (HFE) prediction. Our main goal is to evaluate this framework on out-of- distribution data (molecules that differ significantly from those used in training), where DNNs are known to struggle. Several common physics-based solvation models are used in the evaluation. Graph neural network architectures are tested for their ability to generalize using multiple dataset splits, including out-of-distribution HFEs and unseen molecular scaffolds. Our most important finding is that for out-of-distribution data, where DNNs alone often struggle, the physics + DNN models consistently improve physics model predictions. For in-distribution data, the DNN corrections significantly improve the accuracy of physics-based models, with a final RMSE below 1 kcal/mol and a relative improvement between 40% and 65% in most cases. The accuracy of physics + DNN models tends to improve when the 6% of molecules with the highest experimental uncertainty are removed. This study provides insights into the potential and limitations of combining physics and machine learning for molecular modeling, offering a practical and generalizable strategy.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Machine Learning | <details><summary>Show</summary><p>[{"title": "Supplementary Materials", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683cb33b3ba0887c332eb3dd/original/supplementary-materials.pdf"}]</p></details> |
| **[A graph neural network charge model targeting accurate electrostatic properties of organic molecules](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6839c94c3ba0887c33d2cd8e/original/a-graph-neural-network-charge-model-targeting-accurate-electrostatic-properties-of-organic-molecules.pdf)** | 2025-06-03 | 10.26434/chemrxiv-2025-4tmbz | <details><summary>Show</summary><p>Common methods for assigning atom-centered partial charges in computational chemistry, such as RESP and AM1-BCC, rely on quantum mechanical or semi-empirical calculations of the molecule of interest, which are expensive to compute and dependent on the choice of input molecular conformer(s). Graph neural network (GNN) based continuous atom embeddings have been shown to be a fast and flexible solution for partial charge assignment, but those developed so far for condensed phase modeling have usually been trained to reproduce AM1-BCC charges, which themselves seek to reproduce the HF/6-31G(d) molecular electrostatic potential. Here, we investigate the suitability of various common charge assignment schemes, including ESP and atoms-in-molecule (AIM) based approaches, as training targets for new GNN based charge models. We show that the strengths of both approaches can be combined by co-training GNN models to AIM charges and molecular dipoles and electrostatic potentials. We collect a dataset of quantum mechanical AIM properties computed at a high level of theory (wB97X-D/def2-tzvpp), in both vacuum and implicit solvent, and train new GNN charge models to each. Charges can be scaled between the vacuum and solvated charge sets, and combined with Lennard-Jones parameters optimized using the Open Force Field infrastructure, to yield force fields that are suitably polarized for condensed phase modeling. We further demonstrate that the charge models may be applied to explore electrostatics-driven structure-activity relationships in medicinal chemistry. The charge models are freely available at: https://github.com/cole-group/nagl-mbis/.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Machine Learning; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supporting Information for: A graph neural network charge model targeting accurate electrostatic properties of organic molecules", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6839d2653ba0887c33d3e90f/original/supporting-information-for-a-graph-neural-network-charge-model-targeting-accurate-electrostatic-properties-of-organic-molecules.pdf"}]</p></details> |
| **[A Generalized Machine-Learning Framework for Developing Alchemical Many-Body Interaction Models for Polymer Grafted Nanoparticles](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6838ce27c1cb1ecda034de24/original/a-generalized-machine-learning-framework-for-developing-alchemical-many-body-interaction-models-for-polymer-grafted-nanoparticles.pdf)** | 2025-06-03 | 10.26434/chemrxiv-2025-n4d3s | <details><summary>Show</summary><p>Polymer-grafted nanoparticles (PGNs) serve as highly customizable building blocks for technologically- relevant self-assembled nanomaterials. Physics- informed Inverse design strategies are crucial for ex- pediting exploration of the massive associated design space by identifying optimal PGN attributes such as polymer length and grafting density to meet a target self-assembled structure. However, their success hinges on accurate description of how PGN interactions vary as a function of both particle positions and physical at- tributes. We introduce a framework to meet this need. Specifically, we develop an “alchemical” ML-IAM that describes how PGN interactions vary as a function of both inter-PGN distances and tunable PGN attributes, simultaneously. This model is an extension of the physics-informed and explicitly many-bodied ChIMES model. The resulting extended ChIMES (X-ChIMES) model is trained on PMF data for PGNs with varied polymer ligand length. We enable e cient training data generation by combining the forward-reverse steered molecular dynamics enhanced sampling approach with a grid-sampling scheme in the HOOMD-blue software package. We demonstrate the e cacy of ChIMES for generating CG models for PGNs with fixed design at- tributes, the development of the X-ChIMES model, and the application of the X-ChIMES model that en- hances digital alchemy inverse-design simulations using HOOMD-blue and the ChIMES Calculator. This is the first application of ChIMES in modeling CG systems and coupling with an inverse design method to target nanomaterial self-assembly.</p></details> | Theoretical and Computational Chemistry; Materials Science; Aggregates and Assemblies; Computational Chemistry and Modeling; Machine Learning | <details><summary>Show</summary><p>[]</p></details> |
| **[Transfer learning with domain knowledge adaptation for
stability evaluation of dual-atom catalysts on nitrogen-doped carbon.](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683847073ba0887c33a026a2/original/transfer-learning-with-domain-knowledge-adaptation-for-stability-evaluation-of-dual-atom-catalysts-on-nitrogen-doped-carbon.pdf)** | 2025-06-02 | 10.26434/chemrxiv-2025-pjdpq | <details><summary>Show</summary><p>Dual-atom catalysts are innovative materials with exceptional activity and selectivity. Yet, their stability remains a key challenge in catalyst design. Conventional characterization and synthesis techniques struggle to precisely identify active sites due to their vast configurations, while distinguishing the metal speciation remains challenging. In a theoretical approach, density functional theory based high-throughput screening is constrained by computational cost and time. Herein, we
implemented transfer learning with domain knowledge adaptation for the evaluation of the stability against metal aggregation of DACs on nitrogen-doped carbon. The transferability of the stability descriptors applied to single-atoms on doped carbon to dual-atom catalysts on nitrogen-doped carbon, demonstrates their universality to more complex systems. Valuable insights were gained for
the design of stable catalysts with the identification of the optimum metal pair and coordination environment combination, and the examination of stability-synergistic effect tradeoff.</p></details> | Theoretical and Computational Chemistry; Catalysis; Computational Chemistry and Modeling; Machine Learning; Heterogeneous Catalysis; Materials Chemistry | <details><summary>Show</summary><p>[{"title": "Supplementary information ", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68384e653ba0887c33a186e9/original/supplementary-information.pdf"}]</p></details> |
| **[Circumventing problems introduced by matrix asymmetry in collocation calculations of vibrational spectra by exploiting near symmetry](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683581aac1cb1ecda0ba848b/original/circumventing-problems-introduced-by-matrix-asymmetry-in-collocation-calculations-of-vibrational-spectra-by-exploiting-near-symmetry.pdf)** | 2025-06-02 | 10.26434/chemrxiv-2025-x7wk6 | <details><summary>Show</summary><p>Collocation is an enticing alternative to variational methods for solving the vibrational Schr¨odinger equation. It makes it possible to use a general potential without requiring integrals and quadrature. An important disadvantage of collocation is the need to work with nonsymmetric matrices. Eigenvalues of a large matrix are best computed with an iterative method, but iterative eigensolvers are much more efficient for symmetric matrices. Heretofore, it has been costly to use collocation when the basis set and Hamiltonian matrix are large. We demonstrate that it is possible to systematically make the collocation matrix whose eigenvalues one must compute more and more symmetric and propose an efficient iterative eigensolver for a nearly symmetric matrix. Little is known about exploiting near symmetry. We use a combination of filter diagonalization and an iterative linear solver powered by a three-term recursion relation. We test the ideas with a 6-D Hamiltonian and show that accurate energies are obtained despite the asymmetry.</p></details> | Theoretical and Computational Chemistry; Theory - Computational | <details><summary>Show</summary><p>[]</p></details> |
| **[An Automated QM/MM Average Protein Electrostatic Configuration Approach for Flavoproteins: APEC-F 2.0](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683ead071a8f9bdab561fdcb/original/an-automated-qm-mm-average-protein-electrostatic-configuration-approach-for-flavoproteins-apec-f-2-0.pdf)** | 2025-06-06 | 10.26434/chemrxiv-2025-sp0z3 | <details><summary>Show</summary><p>Flavoproteins are a ubiquitous class of redox proteins, enzymes, and photoreceptors that derive their versatility from the flavin cofactor, a prosthetic group that serves as the main locus of their spectral, photophysical, and (photo)chemical properties. It is thus common for computational modeling of flavoproteins to employ a hybrid approach that treats the flavin quantum mechanically and remaining atoms classically. Such QM/MM methods have proven powerful for studying flavoproteins so far, but users are often faced with a choice between treating the flavin electronic structure with ab initio wave function methods or using more approximate methods that allow for more extensive sampling of the protein dynamics. Herein, we present APEC-F 2.0, an automated QM/MM workflow that uses several open-source software to construct QM/MM models of flavoproteins. Exploiting the rigidity of flavin's tricyclic isoalloxazine ring, the APEC approach iteratively optimizes flavin's geometry in a static MM environment that represents a dynamic protein using a superposition of configurations generated from molecular dynamics. The automation of the code enables the systematic construction of QM/MM models using a common protocol and is suitable for comparing flavin's spectral, electronic, and chemical properties in different redox, protonation, or excited states in a wide range of flavoproteins.</p></details> | Theoretical and Computational Chemistry; Physical Chemistry; Computational Chemistry and Modeling; Theory - Computational; Photochemistry (Physical Chem.) | <details><summary>Show</summary><p>[]</p></details> |
| **[Modular Global Optimization using Molecular Graphs : A Case Study of ZnO/Cu Surface for Methanol Synthesis Reaction](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683e7c92c1cb1ecda0dcda1e/original/modular-global-optimization-using-molecular-graphs-a-case-study-of-zn-o-cu-surface-for-methanol-synthesis-reaction.pdf)** | 2025-06-06 | 10.26434/chemrxiv-2025-t71bx | <details><summary>Show</summary><p>Heterogeneous and electrocatalysts play a crucial role in enabling various industrial chemical transformations, with quantum chemistry calculations serving as a fundamental tool for investigating their atomic-scale properties. Advances in computational power have facilitated the study of increasingly complex catalytic systems, particularly metal-metal oxide interfaces under realistic reaction conditions. However, these studies remain inherently constrained by approximations in computational models, which often fail to fully capture the intricacies of catalytic phenomena. Additionally, the configurational space associated with such systems is too large to be systematically explored using chemical intuition alone. To address these challenges, we introduce GG, a modular graph-based Python package. This approach enables the systematic exploration of configurational space on common catalytic surfaces using molecular graph representations, allowing for efficient scaling to larger systems. We demonstrate the capabilities of GG through a case study for the ZnOxHy/Cu system to gain insights into the active site for methanol synthesis under reaction conditions. The proposed strategy is broadly applicable and can be extended to a wide range of complex atomic systems.</p></details> | Theoretical and Computational Chemistry; Materials Science; Catalysis; Computational Chemistry and Modeling; Theory - Computational; Heterogeneous Catalysis | <details><summary>Show</summary><p>[{"title": "Modular Global Optimization using Molecular Graphs : A Case Study of ZnO/Cu Surface for Methanol Synthesis Reaction", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683e7e771a8f9bdab558a930/original/modular-global-optimization-using-molecular-graphs-a-case-study-of-zn-o-cu-surface-for-methanol-synthesis-reaction.docx"}]</p></details> |
| **[A Multiscale Modeling Approach to Chiral Nanocluster Self-assembly](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683bd9f11a8f9bdab513951e/original/a-multiscale-modeling-approach-to-chiral-nanocluster-self-assembly.pdf)** | 2025-06-04 | 10.26434/chemrxiv-2025-xh721 | <details><summary>Show</summary><p>The controlled formation of chiral microstructures from functionalized nanoparticles remains a key challenge in nanoscience. Factors such as ligand charge, counterion type, and pH critically affect nanoparticle self-assembly, yet molecular-level mechanisms remain poorly understood. In this study, we used atomistic and coarse-grained (CG) molecular dynamics (MD) simulations to investigate the self-assembly of [Ag$_9$(\textit{o}-MBA)$_9$]$^{9-}$ (where \textit{o}-MBA = \textit{ortho}-mercaptobenzoic acid) in the presence of calcium ions. Atomistic MD simulations revealed dynamic silver cores and site-specific Ca$^{2+}$ binding, which promotes the formation of ordered motifs. CG MD simulations enabled us to simulate large multimeric systems with hundreds of nanoclusters over extended timescales. Together, the simulations show that nanoclusters assemble into long linear chains, which subsequently coil into chiral superstructures. These findings provide multiscale, molecular-level insight into how calcium-mediated interactions guide the chiral self-assembly of monolayer-protected metal nanoclusters and establish a computational framework for the rational design of chiral nanomaterials.</p></details> | Theoretical and Computational Chemistry; Nanoscience; Nanostructured Materials - Nanoscience; Computational Chemistry and Modeling; Theory - Computational; Materials Chemistry | <details><summary>Show</summary><p>[{"title": "Supplimentary Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683bdcc93ba0887c3318b198/original/supplimentary-information.pdf"}]</p></details> |
| **[Ruthenium-Catalysed Azide-Selenoalkyne Cycloadditions: A Combined Synthetic-Computational Study into Reaction Scope, Mechanism and Origins of Regioselectivity](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683b63923ba0887c330d5f04/original/ruthenium-catalysed-azide-selenoalkyne-cycloadditions-a-combined-synthetic-computational-study-into-reaction-scope-mechanism-and-origins-of-regioselectivity.pdf)** | 2025-06-04 | 10.26434/chemrxiv-2025-dv1rr-v2 | <details><summary>Show</summary><p>A ruthenium-catalysed azide-alkyne cycloaddition (RuAAC) reaction involving selenoalkynes is reported for the synthesis of selenium-substituted 1,2,3-triazole products (26 examples, up to 89% yield). The reaction works well with a wide range of alkyl and aryl azides bearing both electron-donating and -withdrawing groups. This contrasts RuAAC reactions using regular terminal and internal alkynes, where aryl azides bearing electron-withdrawing groups are not generally tolerated. The regioselectivity of the cycloaddition is highly dependent on the identity of the non-selenium substituent of the alkyne. This challenges earlier reports on RuAAC reactions involving thioalkynes and questions the perceived understanding of the regioselectivity of these reactions. Computational modelling of the reaction profiles for four alkyne substrates that provided contrasting regioselectivities accurately reproduces the experimental results. The observed regioselectivity is primarily attributed to the relative nucleophilicity of the alkyne carbons upon coordination to ruthenium and an approach is proposed in which reaction regioselectivity may be predicted.</p></details> | Theoretical and Computational Chemistry; Organic Chemistry; Catalysis; Organic Synthesis and Reactions; Theory - Computational; Homogeneous Catalysis | <details><summary>Show</summary><p>[{"title": "Supporting Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683b63933ba0887c330d5f15/original/supporting-information.pdf"}]</p></details> |
| **[Analytic gradients and periodic boundary conditions for direct reaction field polarizable QM/MM](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6837117f3ba0887c3374b08c/original/analytic-gradients-and-periodic-boundary-conditions-for-direct-reaction-field-polarizable-qm-mm.pdf)** | 2025-06-02 | 10.26434/chemrxiv-2025-1jrbx | <details><summary>Show</summary><p>Our recently developed Direct Reaction field with ESPF Embedding Model (DREEM) method offers an efficient and physically rigorous framework for incorporating polarizable molecular mechanics (MM) environments into quantum mechanics/molecular mechanics (QM/MM) simulations. By coupling the QM and MM regions through the instantaneous MM electrostatic polarization response to QM charge density fluctuations, DREEM enables consistent treatment of ground and excited electronic states, capturing electronic state-specific polarization and dispersion effects absent in conventional mean-field or linear response approaches. The use of electrostatic potential fitting (ESPF) approximation method to describe charge density fluctuations significantly improves the computational efficiency compared the integral-exact direct reaction field. In this work, we present two methodological advancements to extend the applicability of DREEM to realistic condensed-phase simulations. Firstly the development of efficient analytic energy gradients, enabling geometry optimization, transition state searches, and molecular dynamics, and secondly a formulation of periodic boundary conditions (PBC) compatible with the DREEM framework. These capabilities are implemented in the open-source OpenESPF code, interfacing PySCF and OpenMM for high-performance QM and MM calculations. We demonstrate that the resulting implementation enables practical simulations of excited-state optical properties in periodic polarizable environments, where we calculate the fluorescence spectrum of acetone in water including quantum vibronic and non-Condon effects. This paves the way for predictive modeling of photochemical reactivity and spectroscopy in complex systems where environment polarization is important.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Theory - Computational | <details><summary>Show</summary><p>[{"title": "Supporting Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/683712c0c1cb1ecda0f5ba96/original/supporting-information.pdf"}]</p></details> |

