# Weekly Papers
This project automatically fetches the latest papers from ChemRxiv based on categories.

Only the most recent articles for each category are retained, up to a maximum of 100 papers.

You can click the "Watch" button to receive daily email notifications.

Last update: 2025-09-22

| **Title** | **Date** | **Doi** | **Abstract** | **Categories** | **Suppitems** |
| --- | --- | --- | --- | --- | --- |
| **[Deep Residual Learning for Molecular Force Fields](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c8ff713e708a764967df01/original/deep-residual-learning-for-molecular-force-fields.pdf)** | 2025-09-19 | 10.26434/chemrxiv-2025-003lg-v3 | <details><summary>Show</summary><p>Accurate interatomic interaction modeling via force fields is critical for biological studies and drug discovery, but balancing accuracy and generalizability remains challenging. We introduce ResFF (Residual Learning Force Field), a hybrid machine learning force field that employs deep residual learning to integrate physics-based learnable molecular mechanics covalent terms with residual corrections from a lightweight equivariant neural network. Through a three-stage joint optimization, the two components are trained in a complementary manner to achieve optimal performance. Benchmarks show ResFF outperforms classical and neural network force fields in generalization (mean absolute error (MAE): 1.16 kcal/mol on Gen2-Opt, 0.90 kcal/mol on DES370K), torsional profiles (MAE: 0.45/0.48 kcal/mol on TorsionNet-500 and Torison Scan), and intermolecular interactions (MAE: 0.32 kcal/mol on S66×8). It also enables precise energy minima reproduction and stable molecular dynamics of biological systems. ResFF merges physical constraints with neural expressiveness, offering a robust tool for accurate and efficient molecular simulation.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Machine Learning; Artificial Intelligence | <details><summary>Show</summary><p>[{"title": "Supporting Information of ResFF", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c8ff713e708a764967df14/original/supporting-information-of-res-ff.pdf"}]</p></details> |
| **[A new benchmark for deep learning based affinity prediction: Solving the inter-protein scoring noise problem](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c47cf49008f1a46721c407/original/a-new-benchmark-for-deep-learning-based-affinity-prediction-solving-the-inter-protein-scoring-noise-problem.pdf)** | 2025-09-18 | 10.26434/chemrxiv-2025-sf3cs | <details><summary>Show</summary><p>The incredible advances in deep learning architectures have not only led to successful protein structure prediction methods, but also to many interesting new methods related to protein-ligand docking and co-folding. The most recent biomolecular foundation model, Boltz-2, even claimed to approach the performance of free-energy perturbation (FEP) in estimating small molecule–protein binding affinity.
However, there is still an ongoing debate about the capabilities of such models regarding generalisation or memorisation. A comprehensive assessment remains difficult due to the lack of demanding and relevant benchmark data sets. Here, we present a new approach to benchmark affinity prediction based on the inter-protein scoring noise problem of classical scoring functions. This well-known phenomenon describes the problem that classical scoring functions are capable of enriching active molecules for a protein target but are unable to identify the correct protein target for a given active molecule. This is due to scoring variation between different binding pockets, which limits the ranking capabilities for target identification. In contrast, a successful and reliable binding affinity prediction method should be able to identify the correct protein target and differentiate between binders and non-binders. 
We therefore developed a new benchmark set for target identification based on the LIT-PCBA and tested whether Boltz-2 can correctly identify the target of active molecules. Our study highlights that, while the affinity module of Boltz-2 seems promising, it is still not able to identify the correct protein target by correctly predicting a higher binding affinity in comparison to the decoy targets. This indicates that a generalisation of protein-ligand interactions is still not achieved, which is in line with other benchmarks that indicate a memorizing effect. Such a benchmark can be seen as the next major hurdle to successful deep-learning-based affinity prediction using protein-ligand complexes. Any model or method truly capable of accurate binding affinity prediction should perform well on target-prediction benchmark tasks.
</p></details> | Theoretical and Computational Chemistry; Biological and Medicinal Chemistry; Bioinformatics and Computational Biology; Artificial Intelligence; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supporting Information with additional figures", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c4815523be8e43d6a2259a/original/supporting-information-with-additional-figures.pdf"}, {"title": "Ligands used for Prediction using default settings", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c481773e708a7649b54ba5/original/ligands-used-for-prediction-using-default-settings.csv"}, {"title": "Ligands used for Prediction using Alphafold 3 settings", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c4818623be8e43d6a22b87/original/ligands-used-for-prediction-using-alphafold-3-settings.csv"}]</p></details> |
| **[Impact of chirality on nanotube fracture strain: comprehensive machine learning potential calculations](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c765cc9008f1a46787e869/original/impact-of-chirality-on-nanotube-fracture-strain-comprehensive-machine-learning-potential-calculations.pdf)** | 2025-09-16 | 10.26434/chemrxiv-2025-rzmm5-v2 | <details><summary>Show</summary><p>We present a comprehensive study of fracture strains in carbon nanotubes (CNTs) as a function of their chirality using the direct integration of the exchange potential (DIEP) machine learning potential model. By simulating fracture using quasi-static pulling of the CNT structures, we simulated the fracture of 186 CNTs ranging from 476 atoms up to 5616 atoms in size. Our results reveal clear trends in the fracture strains for armchair, zigzag, and chiral CNTs. In particular, armchair nanotubes tend to exhibit localized bond scission near the ends of the supercell, while zigzag nanotubes suffer more distributed bond breakage along their length. This observation is consistent with previous molecular dynamics simulations. We validate the model's performance by comparing its predicted fracture strains with published empirical model. In addition, we highlight specific CNT configurations with notably high or low fracture strains, shedding light on the interplay between chirality, diameter, and defect propagation mechanisms. The demonstrated predictive power of this machine learning potential underscores its promise for large-scale, high-throughput fracture simulations of CNTs and potentially other nanomaterials.</p></details> | Theoretical and Computational Chemistry; Materials Science; Nanoscience; Artificial Intelligence | <details><summary>Show</summary><p>[]</p></details> |
| **[Enhancing Bioassay Annotations in ChEMBL with Artificial Intelligence](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68bb1ba6a94eede154e9e76f/original/enhancing-bioassay-annotations-in-ch-embl-with-artificial-intelligence.pdf)** | 2025-09-15 | 10.26434/chemrxiv-2025-h4hrr | <details><summary>Show</summary><p>As the volume and diversity of bioactivity data in ChEMBL continues to grow, ensuring that assay metadata is standardized, interoperable, and machine-readable is critical for effective use in cheminformatics and ML applications. In this work, we present recent efforts to enhance the quality and granularity of bioassay annotations in ChEMBL through a combination of manual and semi-manual curation and AI-driven approaches. We introduce a “perfect assay description” template to guide consistent annotation and demonstrate how natural language processing techniques and multi-class classification can be used to automatically extract key assay parameters and assign broad assay categories for legacy data. We report on the development, validation, and application of a spaCy-based NER model that identifies experimental methods with high precision and recall, as well as a complementary classification model that refines ASSAY_TYPE categorization beyond the existing schema. In addition, we describe improvements to metadata extraction for ADME endpoints, organism and protein variant annotations, and ontology linking using tools such as text2term. Together, these enhancements significantly advance the FAIRness of ChEMBL’s bioassay data, enabling more robust downstream analyses and more precise compound-target activity modeling.</p></details> | Theoretical and Computational Chemistry; Artificial Intelligence; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supplementary Material", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68bb2186a94eede154ea9092/original/supplementary-material.pdf"}]</p></details> |
| **[Real-space machine learning of correlation density functionals](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c81b349008f1a467a92116/original/real-space-machine-learning-of-correlation-density-functionals.pdf)** | 2025-09-19 | 10.26434/chemrxiv-2024-zk6hp-v4 | <details><summary>Show</summary><p>Machine learning (ML) plays a pivotal role in extending the reach of quantum chemistry methods for both molecules and materials.  However, leveraging ML to improve upon human-designed density functional approximations (DFAs), the primary workhorse for quantum simulations,  remains challenging due to severely limited transferability to unseen chemical systems.  Here we address this challenge through real-space ML, where energies are learned point by point via correlation energy densities per particle obtained from regularized perturbation theory.  We pursue two crucial strategies that enable the construction of highly transferable DFAs, grounded in the Møller–Plesset adiabatic connection framework, for correlation energies defined with respect to the Hartree–Fock reference. First, we introduce the Local Energy Loss, whose data efficiency (expanding each system’s single energy into thousands of datapoints) dramatically enhances transferability when combined with a physically informed ML construction. Second, we construct a real-space, machine-learned, and regularized extension of Spin-Component-Scaled second-order Møller–Plesset perturbation theory, opening new avenues for developing transferable DFAs particularly suited for overcoming self-interaction errors common to traditional DFAs.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Theory - Computational; Machine Learning | <details><summary>Show</summary><p>[]</p></details> |
| **[Prediction of Charged Small Molecule Conformations in Solution using a Balanced ML/MM Potential](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c6c90a3e708a7649069e12/original/prediction-of-charged-small-molecule-conformations-in-solution-using-a-balanced-ml-mm-potential.pdf)** | 2025-09-18 | 10.26434/chemrxiv-2025-9hps8 | <details><summary>Show</summary><p>Reliably evaluating the most stable conformations of charged small organic molecules in solution poses a major challenge to computational chemistry, due to the accuracy or computational efficiency limitations of conventional methodologies. In this paper, we present a hybrid machine learning/molecular mechanics (ML/MM) potential, with dynamic, conformationally-dependent charges on the atoms in the ML region, that can be used to address this important challenge in drug design. By way of a case study, molecular dynamics simulations were used to compare the performance of intermolecular potentials in evaluating the solution phase conformational free energy differences of pharmaceutically relevant ligands based on the protonated 2-phenylethylamine scaffold. We show that satisfactory performance of an MM-only force field in predicting the preferred conformations of these molecules in solution (with a mean absolute error (MAE) = 0.9 kcal/mol) is due a convenient cancellation of errors in the intramolecular and intermolecular contributions to the conformational free energy difference. Specifically, the underestimation of intramolecular cation- interactions which preferentially stabilise the folded conformation is compensated by underestimated solute-solvent interactions which preferentially stabilise the extended conformation. A straightforward approach to an ML/MM potential, in which the intramolecular MM representation of the solute is simply substituted by a ML model without making other changes to the potential, disrupts this error cancellation and results in a systematic underperformance relative to the MM-only approach, with a MAE of 1.9 kcal/mol. To remedy this shortcoming, a new ML/MM potential is presented that combines an accurate, trained-for-purpose ML (PairFEQ-Net) model of the charged solute, with the polarisable SWM3 MM model of water. Crucially, by employing an empirical parameter to scale gas phase charges, the ML model predicts dynamic, solvent-polarised charges that effectively electrostatically embed the solute in the solvent in a balanced way, outperforming the MM-only approach when it comes to conformational free energy predictions, reducing the MAE to 0.5 kcal/mol. This balanced ML/MM potential furnishes a possible route to the chemically accurate prediction of the shapes of organic molecules in aqueous solution.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Machine Learning | <details><summary>Show</summary><p>[]</p></details> |
| **[B5 Active-Site Design in Co–Cu–Fe–Mo–Ni High-Entropy Alloys for Ammonia Decomposition](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c6adf63e708a764902cfe4/original/b5-active-site-design-in-co-cu-fe-mo-ni-high-entropy-alloys-for-ammonia-decomposition.pdf)** | 2025-09-18 | 10.26434/chemrxiv-2025-f5jsl | <details><summary>Show</summary><p>Co–Cu–Fe–Mo–Ni high-entropy alloys (HEAs) show strong activity for ammonia decomposition reaction (ADR). Owing to the vast composition space and varied surface structures, traditional investigation methods for HEAs meet great challenges. We develop a DFT-trained cluster expansions (CE) model coupled with a microkinetic volcano plot and Metropolis Monte Carlo (MMC) annealing to link composition, B5 sites on fcc(211), *N-adsorption (ΔE(*N)), and turnover frequency (TOF). CE model predicts site-resolved ΔE(*N), enabling maps of site-specific TOF and surface-averaged activity (⟨TOF⟩) across the quinary space. MMC reveals temperature-driven Cu surface enrichment that shifts ΔE(*N) distribution toward weaker binding relative to fully random surfaces, thereby suppressing ⟨TOF⟩. Lowering Cu content systematically increases activity; by contrast, Cu-free Co–Fe–Mo–Ni medium-entropy alloys (MEAs) exhibit ΔE(*N) distributions clustered near the volcano maximum and deliver high ADR activity with limited sensitivity to Co/Fe/Ni composition swaps. Site-level analysis shows that the most active B5 sites contains less Cu, consistent with surface-averaged trends. DFT validation on a curated set of B5 sites confirms the accuracy of CE-screened high-activity sites. The workflow provides practical design rules—minimize Cu participation at B5 and preserve configurational disorder—and is readily extensible to other alloy families and catalytic reactions. </p></details> | Theoretical and Computational Chemistry; Catalysis; Energy; Computational Chemistry and Modeling; Machine Learning; Fuels - Energy Science | <details><summary>Show</summary><p>[]</p></details> |
| **[Accurate Hydration Free Energy Calculations for Diverse Organic Molecules With a Machine Learning Force Field](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c48aa423be8e43d6a4015e/original/accurate-hydration-free-energy-calculations-for-diverse-organic-molecules-with-a-machine-learning-force-field.pdf)** | 2025-09-18 | 10.26434/chemrxiv-2025-p7r0r | <details><summary>Show</summary><p>Free energy perturbation (FEP) calculations using classical force fields remain the dominant approach for large-scale, computational drug discovery efforts but the accuracy is fundamentally limited by simplified forms that cannot quantitatively reproduce ab initio methods without significant fine tuning.  Machine Learning force fields (MLFFs) offer a promising avenue to retain quantum mechanical accuracy with significantly reduced computational cost compared to ab initio molecular dynamics (AIMD) simulations. Thus far, direct applications of ML force fields to FEP calculations lack systematic protocols and extensive benchmarking. In this work, we take a step in this direction by presenting a general and robust workflow for solvation (hydration) free energy (HFE) calculations which is independent of the details of the particular MLFF architecture used. Combining a broadly trained ML force field, Organic_MPNICE, with sufficient statistical and conformational sampling empowered by the solute-tempering technique, affords sub-kcal/mol average errors in HFE predictions relative to experimental estimates.  This approach outperforms state-of-the-art classical force fields and DFT-based implicit solvation models on a diverse set of 59 organic molecules and provides a route to ab initio-quality HFE predictions, advancing the use of ML force fields in thermodynamic property prediction.</p></details> | Theoretical and Computational Chemistry; Physical Chemistry; Organic Chemistry; Computational Chemistry and Modeling; Machine Learning; Physical and Chemical Properties | <details><summary>Show</summary><p>[{"title": "Supplementary Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c48c099008f1a46724973f/original/supplementary-information.pdf"}]</p></details> |
| **[Evidence for significant multi-Li+ clustering in common lithium-ion battery electrolytes](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c08e353e708a7649010b9a/original/evidence-for-significant-multi-li-clustering-in-common-lithium-ion-battery-electrolytes.pdf)** | 2025-09-17 | 10.26434/chemrxiv-2025-ndqzk | <details><summary>Show</summary><p>Lithium-ion battery (LIB) electrolytes have been empirically optimized to maximize battery lifetime and charge rate, but a detailed understanding of how their atomic-scale structure controls their electrochemical behaviors remains lacking. Here, we combine machine-learning-enabled atomistic simulations, nuclear magnetic resonance measurements, electrochemical data, and quantum-chemical calculations to reveal previously unstudied multi-Li+ complexes without anions that are uniquely facile for reduction. This surprising insight explains the success of empirically-chosen solvent mixtures and provides a foundation for the rational design of improved electrolytes. Furthermore, our machine learning force field (MLFF) based LIB electrolyte simulations accurately reproduce a wide range of condensed phase experiments, despite being parameterized only to quantum chemical gas phase cluster calculations, and hence provide a compelling demonstration that MLFF models developed from first principles can robustly represent a complex chemical system to atomic resolution.</p></details> | Theoretical and Computational Chemistry; Energy; Computational Chemistry and Modeling; Machine Learning; Energy Storage; Materials Chemistry | <details><summary>Show</summary><p>[]</p></details> |
| **[Equivariant Neural Networks Reveal How Host–Guest Interactions Shape 129Xe NMR in Porous Liquids](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c42a0323be8e43d6976d8b/original/equivariant-neural-networks-reveal-how-host-guest-interactions-shape-129xe-nmr-in-porous-liquids.pdf)** | 2025-09-17 | 10.26434/chemrxiv-2025-njln9 | <details><summary>Show</summary><p>Understanding host-guest interactions in porous liquids (PLs) formed from porous organic cages (POCs) is pivotal in tailoring their physicochemical properties, therefore, providing an avenue for engineering new PLs with enhanced functionalities. In this work, we demonstrate, for the first time, the use of an accurate and efficient machine learning-based approach for atomistic modeling of host-guest interactions in large-scale PLs. The approach uses E(3)-equivariant graph neural networks (EGNNs) to construct a machine learning interatomic potential and a nuclear magnetic resonance machine learning model. The former enables machine learning molecular dynamics (MLMD) simulations, while the latter computes 129Xe isotropic chemical shift, δiso, from MLMD snapshots. Applied to a PL composed of CC3 POC in 4-(trifluoromethoxy)benzyl alcohol (TBA) solvent loaded with high Xe concentration, this  dual-model approach show that host(CC3)–guest(Xe) interactions are best described by a three-site binding model comprising the CC3 cavity, CC3 openings, and TBA, with exchange events occurring between these sites. Good agreement between computed and experimental 129Xe δiso validate our approach, demonstrating EGNN-based simulations as transformative tools for advancing PL understanding.</p></details> | Theoretical and Computational Chemistry; Physical Chemistry; Computational Chemistry and Modeling; Machine Learning; Materials Chemistry | <details><summary>Show</summary><p>[{"title": "Supporting Information for \"Equivariant Neural Networks Reveal How Host\u2013Guest Interactions Shape 129Xe NMR in Porous Liquids\"", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c42cf93e708a7649ab998d/original/supporting-information-for-equivariant-neural-networks-reveal-how-host-guest-interactions-shape-129xe-nmr-in-porous-liquids.pdf"}]</p></details> |
| **[Rapid prediction of adsorbate probability distributions in metal-organic frameworks using graph neural networks](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68bf29fd23be8e43d6c7af1f/original/rapid-prediction-of-adsorbate-probability-distributions-in-metal-organic-frameworks-using-graph-neural-networks.pdf)** | 2025-09-17 | 10.26434/chemrxiv-2025-tn0rh | <details><summary>Show</summary><p>Metal-organic frameworks (MOFs) are porous crystalline materials assembled from inorganic nodes and inorganic linkers. These materials have garnered significant interest for gas separation and storage applications, particularly because of their porosity and their tunability due to their massive design space. However, navigating such a massive design space poses significant challenges. Atomistic simulation techniques have been applied to accelerate discovery and design of MOFs for various applications. A key property obtained from these simulations is the adsorbate probability distribution (APD). An APD maps the probability of finding an adsorbate molecule in the pore of a MOF at a given temperature and pressure, whose maxima correspond to free energy minima (i.e., binding sites). While APDs and binding sites are not easily accessible experimentally, their generation via simulation is tractable. However, high-throughput generation of APDs still requires long simulation times to converge. A machine learning (ML) model to predict APDs would enable the use of this property in data-driven pipelines to identify high performing materials or binding sites. To date, nobody has attempted to apply ML to the prediction of APDs or binding sites of MOFs. 
In this work, we present DeepAPD – a ML model which predicts APDs at a given temperature and pressure. As an initial proof of concept, the model has been trained on simple spherical adsorbates such as CH4 and Xe. DeepAPD was found to generate APDs of MOFs at a speedup factor of >105 in comparison to GCMC. An in-depth discussion of training strategies and dataset size/composition on model performance is presented. It was found that the APDs obtained by ML were sufficiently accurate to get a reliable estimation of binding sites in MOFs, particularly binding sites which have high probability. Finally, the transferability of the ML models was investigated by evaluating the performance of the GNN model on a dataset of experimentally characterized MOFs. We have also implemented the DeepAPD inference code into our binding site identification algorithm to facilitate an end-to-end MOF to binding site prediction. Future work will extend these models to more complex guests such as CO2, N2, and H2O.</p></details> | Theoretical and Computational Chemistry; Physical Chemistry; Computational Chemistry and Modeling; Machine Learning; Materials Chemistry | <details><summary>Show</summary><p>[{"title": "Supplementary Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68bf2b7223be8e43d6c7d89f/original/supplementary-information.pdf"}]</p></details> |
| **[Beyond conventional sequence-based encoding: a generalized Machine Learning pipeline for oligonucleotide property prediction](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c2e37523be8e43d6658989/original/beyond-conventional-sequence-based-encoding-a-generalized-machine-learning-pipeline-for-oligonucleotide-property-prediction.pdf)** | 2025-09-16 | 10.26434/chemrxiv-2025-nfs45 | <details><summary>Show</summary><p>The development of RNA-based therapeutics has significantly expanded the landscape of drug discovery by enabling precise modulation of gene expression. These approaches offer the potential to target previously "undruggable" genes, overcoming limitations inherent to traditional small molecule therapies. However, the rational design of effective oligonucleotides remains challenging partially due to the limitations of existing computational models which rely predominantly on sequence-based descriptors. These models are generally restricted to naturally occurring oligonucleotides and do not account for the chemical modifications that are essential for enhancing pharmacological and pharmacokinetic properties. To address this gap, we propose a novel chemical-based representation of oligonucleotides exploiting SMILES notation and molecular fingerprints, mainly exploited for small molecules. By applying them to both natural and chemically modified oligonucleotides, we trained machine learning (ML) models that achieved predictive performance comparable to state-of-the-art sequence-based approaches. Furthermore, we curated the first-ever collection of chemically modified oligonucleotides from patent literature, that allowed us to highlight the broad applicability and generalizability of our approach. This work lays the foundation for a unified pipeline capable of modeling both natural and modified oligonucleotides, advancing the computational design of next-generation RNA therapeutics.</p></details> | Theoretical and Computational Chemistry; Biological and Medicinal Chemistry; Computational Chemistry and Modeling; Machine Learning; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "supporting methods", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c2ec313e708a76497999e3/original/supporting-methods.pdf"}, {"title": "Huesken_19nt", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c2ec9f9008f1a467e71716/original/huesken-19nt.csv"}, {"title": "Harborth_19nt", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c2ecfd3e708a764979abd4/original/harborth-19nt.csv"}, {"title": "Reynolds_19nt", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c2ed313e708a764979b2f9/original/reynolds-19nt.csv"}, {"title": "Shabalina_19nt", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c2ed4e3e708a764979b7ce/original/shabalina-19nt.csv"}, {"title": "Vickers_19nt", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c2eecd3e708a764979e42e/original/vickers-19nt.csv"}, {"title": "WO2022159158A1", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c2eefa23be8e43d6670114/original/wo2022159158a1.csv"}]</p></details> |
| **[Mechanism-Oriented Reaction Descriptors for Dispersity Prediction in Nitroxide-Mediated Radical Polymerization Considering Mediator Stability](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c0c76b3e708a764909e9ed/original/mechanism-oriented-reaction-descriptors-for-dispersity-prediction-in-nitroxide-mediated-radical-polymerization-considering-mediator-stability.pdf)** | 2025-09-15 | 10.26434/chemrxiv-2025-zbsh9 | <details><summary>Show</summary><p>In recent years, there has been tremendous growth in data-driven approaches in the field of polymer science, mainly using large databases. However, these analyses do not often lead to mechanism-based interpretations, such as validating a hypothesis for a key reaction phenomenon. In this study, quantitative structure-property relationship (QSPR) models for dispersity (Đ) are constructed using reaction representations based on a putative reaction mechanism. These models are built on nitroxide-mediated radical polymerization reaction data sets collected from the literature. The proposed reaction representation consists of 17 descriptors based on the quasi-equilibrium reaction of forming dormant species, including the stability of mediator radicals. The tendency for the decomposition of the mediator radicals is quantified with a single descriptor. Excluding an unstable mediator radical based on this descriptor significantly increases the prediction accuracy for a poly(n-butyl acrylate) data set. Removing reactions proceeding on different reaction paths contributes to more accurate QSPR-based Đ prediction models, and also to clearer model interpretation in light of the mechanism assumption. Additionally, prediction is found to be possible for data sets from different research groups under similar experimental conditions, while no meaningful prediction is achieved for reactions conducted under different experimental conditions and monomer conversion.</p></details> | Theoretical and Computational Chemistry; Polymer Science; Polymerization (Polymers); Machine Learning; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supporting Information ", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c0d34e23be8e43d6f86d20/original/supporting-information.pdf"}]</p></details> |
| **[A Versatile Method for Accurately Predicting Electronic Absorption Spectra of Tetrapyrrole Macrocycles](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68be9550728bf9025e06af56/original/a-versatile-method-for-accurately-predicting-electronic-absorption-spectra-of-tetrapyrrole-macrocycles.pdf)** | 2025-09-18 | 10.26434/chemrxiv-2025-9gsp2 | <details><summary>Show</summary><p>Understanding the UV-Vis-NIR electronic spectra of tetrapyrrole macrocycles (porphyrins, chlorins, and bacteriochlorins) is crucial to rational designs of novel functional molecules and materials for energy conversion, catalysis, bio-imaging, and medical applications. Despite this, no methods have been defined to accurately predict the electronic absorption spectra of tetrapyrrole molecules bypassing the expensive and time-consuming TDDFT calculations and enable high throughput screening for thousands of molecules in a short-time period. Using a combination of computational chemistry, Gouterman four-orbital model, and machine learning methods, we show accurate predictions of the electronic absorption spectra of tetrapyrroles bearing various peripheral substituents across the UV-Vis-NIR spectral regions. Optimized AutoGluon pipelines trained on the full data set achieve high predictive fidelity (R² = 0.985 validation, 0.936 test; mean/median errors < 0.1 eV) and robust transferability across different macrocyclic scaffolds. Together, this work establishes an efficient (cost and time), scalable, and mechanistically grounded framework for predicting tetrapyrrole electronic absorption spectra, offering broad utilities in rational designs of tetrapyrrole-based functional materials for real-world applications.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Theory - Computational; Quantum Computing | <details><summary>Show</summary><p>[{"title": "Supporting Information: Predicting Electronic Absorption Spectra of Tetrapyrrole Macrocycles", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68bea9bba94eede154536f3a/original/supporting-information-predicting-electronic-absorption-spectra-of-tetrapyrrole-macrocycles.pdf"}]</p></details> |
| **[CRISP: Enhancing ASE Workflows with Advanced Molecular Simulation Post-Processing](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c93cc09008f1a467de5377/original/crisp-enhancing-ase-workflows-with-advanced-molecular-simulation-post-processing.pdf)** | 2025-09-19 | 10.26434/chemrxiv-2025-dht8g-v2 | <details><summary>Show</summary><p>Molecular simulations are invaluable for analysing molecular systems, but existing post-processing tools are often limited by a lack of customisation, interactivity, and efficiency with large datasets. To address this, we developed CRISP (Comprehensive Repository for Insightful Simulation Post-Processing), an open-source Python toolkit designed to enhance workflows within the Atomic Simulation Environment (ASE). CRISP provides a versatile platform for detailed analysis and visualisation, featuring a customizable and modular design, various static analysis methods, interactive 3D visualisations, and parallel processing capabilities optimised for high-performance computing. We demonstrate its effectiveness through case studies, including the analysis of statistical convergence in zeolite simulations, subsampling large datasets for machine learning, and analysing the dynamic stability of atomic clusters. CRISP effectively bridges the gap between raw simulation data and actionable insights, offering an efficient solution for researchers and saving significant time in code development. </p></details> | Theoretical and Computational Chemistry; Theory - Computational; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supplementary Information of, CRISP: Enhancing ASE Workflows with Advanced Molecular Simulation Post-Processing", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c93cc09008f1a467de5383/original/supplementary-information-of-crisp-enhancing-ase-workflows-with-advanced-molecular-simulation-post-processing.pdf"}]</p></details> |
| **[A natural scheme for the quantitative analysis of the magnetically induced molecular current density using an oriented flux-weighted stagnation graph. II. Stagnation line branching and homodiatomics](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c81c559008f1a467a948a8/original/a-natural-scheme-for-the-quantitative-analysis-of-the-magnetically-induced-molecular-current-density-using-an-oriented-flux-weighted-stagnation-graph-ii-stagnation-line-branching-and-homodiatomics.pdf)** | 2025-09-19 | 10.26434/chemrxiv-2025-v4qh1 | <details><summary>Show</summary><p>We have previously introduced a new “natural” scheme to quantitatively analyze the magnetically induced molecular current density vector field, $J^B$, via the oriented flux-weighted stagnation graph (OFW-SG). In this work, we exemplify the procedure via the an- alytic examples of (a) one single vortex, (b) a superposition of two vortices, and (c) a poloidal vortex, and subsequently, we investi- gate a series of homodiatomic molecules, where, in addition to the features of LiH we reported about in the first paper of this series, instances of the current vortex configuration (b) are occurring. We call this configuration “single stagnation line branching” and give a topological and quantitative current density analysis using OFW- SGs.</p></details> | Theoretical and Computational Chemistry; Theory - Computational | <details><summary>Show</summary><p>[]</p></details> |
| **[A semi-focused multi-state variant of the mapping approach to surface hopping](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c4123d3e708a7649a43952/original/a-semi-focused-multi-state-variant-of-the-mapping-approach-to-surface-hopping.pdf)** | 2025-09-17 | 10.26434/chemrxiv-2025-9tg9g | <details><summary>Show</summary><p>The mapping approach to surface hopping (MASH) is one of the most promising methods for simulating nonadiabatic dynamics in molecular systems, in a mixed quantum/classical framework. In its original formulation, MASH is limited to the treatment of two-state systems. Here, we present a generalization of MASH to multiple electronic states, which we call semi-focused MASH (SMASH). A key distinguishing aspect of our approach is that only a selected subset of electronic states, identified through an appropriate clustering procedure, is initially populated. Test simulations of the ultrafast photodynamics of three molecular systems (spiropyran, thioguanine, and azobenzene) show that SMASH gives results closely matching those of decoherence-corrected fewest-switches surface hopping, while eliminating the need for any ad hoc corrections.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Theory - Computational | <details><summary>Show</summary><p>[{"title": "Supporting Information: A semi-focused multi-state variant of the mapping approach to surface hopping", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c415b13e708a7649a523ca/original/supporting-information-a-semi-focused-multi-state-variant-of-the-mapping-approach-to-surface-hopping.pdf"}]</p></details> |
| **[Modeling chain relaxation in moderately entangled polyisoprene melts](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c43bc923be8e43d69a8fbf/original/modeling-chain-relaxation-in-moderately-entangled-polyisoprene-melts.pdf)** | 2025-09-16 | 10.26434/chemrxiv-2025-jv839-v2 | <details><summary>Show</summary><p>Chain relaxation in melts of moderately entangled macromolecules presents the first signs of topological constraints that are commonly manifested in the rheology of long commercial- grade polymers. In this study, we employ multiscale molecular dynamics simulations to examine the relaxation characteristics of model polyisoprene (PI) melts through the transition from the Rouse regime to entangled systems. We probed the impact of molecular weight and stereochemistry, and we evaluated the accuracy of theory in modeling the end-to-end relaxation. Our results support that trans-PI dynamics is distinct, with entanglements forming at a lower degree of polymerization than other compositions due to differences in the characteristic ratio and packing of this polymer. For all moderately entangled systems, modeling the end- to-end relaxation with the original Likhtman-McLeish theory with contour-length fluctuations presents deviations from recorded data both in the early-time and terminal regime. We propose numerical modifications that significantly improve the model for all systems studied without introducing additional parameters. Our approach allows accurate predictions for long macromolecules based on data from detailed simulations with moderately entangled polyisoprene systems.</p></details> | Theoretical and Computational Chemistry; Polymer Science; Chemical Engineering and Industrial Chemistry; Polymer chains; Theory - Computational; Transport Phenomena (Chem. Eng.) | <details><summary>Show</summary><p>[{"title": "Supplemental Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c43bc923be8e43d69a8fcd/original/supplemental-information.pdf"}]</p></details> |
| **[On the role of electronic correlation and state-specific environment polarization in Singlet–Triplet gap inversion](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c3f2c39008f1a467078b4d/original/on-the-role-of-electronic-correlation-and-state-specific-environment-polarization-in-singlet-triplet-gap-inversion.pdf)** | 2025-09-16 | 10.26434/chemrxiv-2025-cgx8x-v2 | <details><summary>Show</summary><p>Molecules characterized by an inverted singlet-triplet gap (ΔEST < 0) hold potential for optoelectronic applications. Electronic correlation and environmental polarization are key factors influencing negative ΔEST , and the latter is gaining attention for its possible role in “mimicking” correlation contributions to yield negative ΔEST . However, a comprehensive study of solvation effects on both structures and energy gaps is still lacking. In this work, we evaluate computational strategies for calculating ΔEST < 0 gaps, incorporating electronic correlation and solvent polarization in molecules exhibiting singlet-triplet inversion. Using RMS–CASPT2 as a benchmark, we demonstrate that double-hybrid density functionals and mixed-reference spin-flip TD-DFT (MRSF– TD-DFT) can partially recover electronic correlation. Furthermore, we investigate solvation effects on both singlet and triplet excited states, highlighting the limitations of linear-response schemes in continuum solvation models. We finally develop a protocol combining electronic correlation and state-specific solvent polarization using double-hybrid functionals and the Vertical Excitation Model (VEM), leveraging its Lagrangian implementation to compute structures and adiabatic energies. Applying our B2PLYP/VEM(UD) protocol to larger systems with experimentally observed negative ΔEST gaps, we quantitatively reproduce experimental emissive and non-radiative transition rates.</p></details> | Theoretical and Computational Chemistry; Energy; Computational Chemistry and Modeling; Theory - Computational; Materials Chemistry | <details><summary>Show</summary><p>[{"title": "Supporting Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c3f2c39008f1a467078b5e/original/supporting-information.pdf"}]</p></details> |
| **[Understanding DFT uncertainties for more reliable reactivity predictions by advancing the analysis of error sources](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c1be759008f1a4679df3d3/original/understanding-dft-uncertainties-for-more-reliable-reactivity-predictions-by-advancing-the-analysis-of-error-sources.pdf)** | 2025-09-16 | 10.26434/chemrxiv-2025-jdfq2-v2 | <details><summary>Show</summary><p>Decades of advancements and thousands of successful applications contributed to the reliability of density functional theory (DFT) methods. Especially in main group chemistry, DFT predictions tend to be increasingly more predictive. In this study we deeply analyze unexpected (ca. 8–13 kcal/mol) DFT disagreements obtained for a few organic reactions using only widely adopted, modern, hybrid and higher rung DFT methods. To understand the underlying causes, here, we move beyond conventional statistics-based benchmarks by combining recent advances in DFT error decomposition with affordable gold-standard references. This approach helps to characterize and disentangle multiple functional and density-based error types and enables us to find functional(s) suitable for broad mechanistic studies in all studied examples. The proposed tools are cost-efficient, readily accessible, and easy to integrate into routine thermochemistry workflows. While the focus is on main group reactions, the approach is applicable also for transition metal, bio-, and surface chemistry to assist more predictive reactivity modeling.</p></details> | Theoretical and Computational Chemistry; Organic Chemistry; Catalysis; Computational Chemistry and Modeling; Theory - Computational; Organocatalysis | <details><summary>Show</summary><p>[{"title": "Additional DFT and CCSD(T) results, extended DFT analysis", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c1be769008f1a4679df3dd/original/additional-dft-and-ccsd-t-results-extended-dft-analysis.pdf"}]</p></details> |
| **[SQUIRREL: An Open-Source Software Suite for Quantum Dynamics Calculations on Complex Geometries with Time-dependent Electric/Magnetic Fields](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c11e389008f1a467816bd7/original/squirrel-an-open-source-software-suite-for-quantum-dynamics-calculations-on-complex-geometries-with-time-dependent-electric-magnetic-fields.pdf)** | 2025-09-15 | 10.26434/chemrxiv-2025-kxmzm | <details><summary>Show</summary><p>We present a general-purpose, open-source software suite, SQUIRREL (Streamlined Quantum Unified Interface for Researching Real-time Excitations with Light), for propagating the time-dependent Schrödinger equation on complex geometries in the presence of time-dependent electric and/or magnetic fields. To handle large systems that can be executed on a conventional desktop computer, the SQUIRREL software suite uses a suite of efficient propagation methods for various quantum dynamics applications, including a new perturbation-based element-dropping algorithm that improves computational performance with minimal loss of accuracy. We analyze the efficacy of these optimizations for Crank-Nicolson, scaled Taylor series approximation, and split-operator propagation methods and discuss the range of their applicability to a variety of quantum dynamics problems. In addition, we provide several examples of time-dependent dynamics calculations and extensive documentation for generating custom geometries, potentials, and time-propagation approaches. Our numerical benchmarks and results demonstrate the versatility of the SQUIRREL software suite for efficiently calculating quantum dynamics in complex nanoscale geometries, particularly in the presence of time-dependent magnetic fields, which have received less attention in previous quantum dynamics studies.</p></details> | Theoretical and Computational Chemistry; Physical Chemistry; Materials Science; Optical Materials; Theory - Computational; Quantum Mechanics | <details><summary>Show</summary><p>[{"title": "Supplemental Materials", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68c121a89008f1a46781e456/original/supplemental-materials.pdf"}]</p></details> |

